gfx/UltimateCataclysmDemo/tile_config.json

```diff
@@ -1 +1 @@
ile": true, "additional_tiles": [{"id": "broken", "fg": 2430}]}, {"id": "vp_woodhalfboard_ne", "fg": 2443, "rotates": true}, {"id": "vp_stowboard_nw", "fg": 2541, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2556}]}, {"id": "vp_door_sliding", "fg": 2584, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2507}, {"id": "broken", "fg": 2487}]}, {"id": "vp_hdframe_nw", "fg": 2483, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2472}]}, {"id": "vp_hdhalfboard_ne", "fg": 2382, "rotates": true}, {"id": "[vp_xlhalfboard_nw],[vp_fxlhalfboard_nw]", "fg": 2385, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2389}]}], "//": "range 2369 to 2608"}, {"file": "opengameartgiant.png", "tiles": [{"id": "t_tree_chestnut", "fg": 2610, "bg": 2266}, {"id": "t_tree_chestnut_season_summer", "fg": 2613, "bg": 2265}, {"id": "t_tree_chestnut_season_autumn", "fg": 2613, "bg": 2264}, {"id": "t_tree_chestnut_season_winter", "fg": 2613, "bg": []}, {"id": "t_tree_pine", "fg": 2614, "bg": 2266}, {"id": "t_tree_pine_season_summer", "fg": 2614, "bg": 2265}, {"id": "t_tree_pine_season_autumn", "fg": 2614, "bg": 2264}, {"id": "t_tree_pine_season_winter", "fg": 2614, "bg": []}, {"id": "t_tree_deadpine", "fg": 2612, "bg": 2266}, {"id": "t_tree_deadpine_season_summer", "fg": 2612, "bg": 2265}, {"id": "t_tree_deadpine_season_autumn", "fg": 2612, "bg": 2264}, {"id": "t_tree_deadpine_season_winter", "fg": 2612, "bg": []}, {"id": "t_tree_hickory", "fg": 2611, "bg": 2266}, {"id": "t_tree_hickory_season_summer", "fg": 2611, "bg": 2265}, {"id": "t_tree_hickory_season_autumn", "fg": 2611, "bg": 2264}, {"id": "t_tree_hickory_season_winter", "fg": 2609, "bg": []}, {"id": "t_tree_hickory_dead", "fg": 2609, "bg": 2266}, {"id": "t_tree_hickory_dead_season_summer", "fg": 2609, "bg": 2265}, {"id": "t_tree_hickory_dead_season_autumn", "fg": 2609, "bg": 2264}, {"id": "t_tree_hickory_dead_season_winter", "fg": 2609, "bg": []}], "//": "range 2609 to 2624", "sprite_width": 96, "sprite_height": 96, "sprite_offset_x": -32, "sprite_offset_y": -64}, {"file": "fallback.png", "tiles": [], "ascii": [{"offset": 0, "bold": false, "color": "BLACK"}, {"offset": 256, "bold": true, "color": "WHITE"}, {"offset": 512, "bold": false, "color": "WHITE"}, {"offset": 768, "bold": true, "color": "BLACK"}, {"offset": 1024, "bold": false, "color": "RED"}, {"offset": 1280, "bold": false, "color": "GREEN"}, {"offset": 1536, "bold": false, "color": "BLUE"}, {"offset": 1792, "bold": false, "color": "CYAN"}, {"offset": 2048, "bold": false, "color": "MAGENTA"}, {"offset": 2304, "bold": false, "color": "YELLOW"}, {"offset": 2560, "bold": true, "color": "RED"}, {"offset": 2816, "bold": true, "color": "GREEN"}, {"offset": 3072, "bold": true, "color": "BLUE"}, {"offset": 3328, "bold": true, "color": "CYAN"}, {"offset": 3584, "bold": true, "color": "MAGENTA"}, {"offset": 3840, "bold": true, "color": "YELLOW"}]}]}
```

Comment:
```suggestion
{"tile_info": [{"width": 32, "height": 32}], "tiles-new": [{"file": "normal.png", "tiles": [{"id": "t_wall", "multitile": true, "fg": 14, "additional_tiles": [{"id": "center", "fg": 9}, {"id": "corner", "fg": [6, 1, 15, 7]}, {"id": "t_connection", "fg": [13, 12, 10, 16]}, {"id": "edge", "fg": [4, 3]}, {"id": "end_piece", "fg": [2, 8, 5, 11]}, {"id": "unconnected", "fg": 14}]}, {"id": ["t_water_moving_dp", "t_swater_moving_dp"], "multitile": true, "fg": 28, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 30}, {"id": "corner", "bg": 938, "fg": [31, 18, 19, 20]}, {"id": "t_connection", "bg": 938, "fg": [24, 17, 25, 32]}, {"id": "edge", "bg": 938, "fg": [29, 27]}, {"id": "end_piece", "bg": 938, "fg": [22, 26, 21, 23]}, {"bg": 938, "id": "unconnected", "fg": 28}]}, {"id": ["t_water_moving_dp_season_summer", "t_swater_moving_dp_season_summer"], "multitile": true, "fg": 28, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 30}, {"id": "corner", "bg": 937, "fg": [31, 18, 19, 20]}, {"id": "t_connection", "bg": 937, "fg": [24, 17, 25, 32]}, {"id": "edge", "bg": 937, "fg": [29, 27]}, {"id": "end_piece", "bg": 937, "fg": [22, 26, 21, 23]}, {"bg": 937, "id": "unconnected", "fg": 28}]}, {"id": ["t_water_moving_dp_season_autumn", "t_swater_moving_dp_season_autumn"], "multitile": true, "fg": 28, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 30}, {"id": "corner", "bg": 946, "fg": [31, 18, 19, 20]}, {"id": "t_connection", "bg": 946, "fg": [24, 17, 25, 32]}, {"id": "edge", "bg": 946, "fg": [29, 27]}, {"id": "end_piece", "bg": 946, "fg": [22, 26, 21, 23]}, {"bg": 946, "id": "unconnected", "fg": 28}]}, {"id": ["t_water_moving_dp_season_winter", "t_swater_moving_dp_season_winter"], "multitile": true, "fg": 28, "bg": 587, "additional_tiles": [{"id": "center", "bg": 587, "fg": 30}, {"id": "corner", "bg": 587, "fg": [31, 18, 19, 20]}, {"id": "t_connection", "bg": 587, "fg": [24, 17, 25, 32]}, {"id": "edge", "bg": 587, "fg": [29, 27]}, {"id": "end_piece", "bg": 587, "fg": [22, 26, 21, 23]}, {"bg": 587, "id": "unconnected", "fg": 28}]}, {"id": "t_shrub", "fg": 33, "bg": 938}, {"id": "t_shrub_season_summer", "fg": 33, "bg": 937}, {"id": "t_shrub_season_autumn", "fg": 33, "bg": 946}, {"id": "t_shrub_season_winter", "fg": 34, "bg": 586}, {"id": "t_curtains", "fg": 35}, {"id": ["t_gates_mech_control"], "fg": 39, "bg": 353}, {"id": ["t_gates_mech_control_lab"], "fg": 37, "bg": 353}, {"id": ["t_gates_control_concrete"], "fg": 38, "bg": 825}, {"id": ["t_gates_control_concrete_lab"], "fg": 37, "bg": 2273}, {"id": ["t_gates_control_brick"], "fg": 38, "bg": 576}, {"id": ["t_gates_control_brick_lab"], "fg": 37, "bg": 576}, {"id": ["t_gates_control_metal"], "fg": 38, "bg": 2287}, {"id": ["t_gates_control_metal_lab"], "fg": 37, "bg": 2287}, {"id": ["t_elevator_control"], "fg": 36, "bg": 353}, {"id": ["t_elevator_control_off"], "fg": 40, "bg": 353}, {"id": "t_carpet_red", "multitile": true, "fg": 51, "additional_tiles": [{"id": "center", "fg": 44}, {"id": "corner", "fg": [47, 41, 43, 48]}, {"id": "t_connection", "fg": [52, 45, 56, 54]}, {"id": "edge", "fg": [46, 55]}, {"id": "end_piece", "fg": [53, 50, 42, 49]}, {"id": "unconnected", "fg": 51}]}, {"id": "t_grass", "multitile": true, "fg": 75, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": [{"weight": 1, "sprite": 74}, {"weight": 1, "sprite": 71}, {"weight": 1, "sprite": 66}, {"weight": 1, "sprite": 58}, {"weight": 1, "sprite": 61}, {"weight": 1, "sprite": 73}, {"weight": 1, "sprite": 59}, {"weight": 1, "sprite": 68}, {"weight": 1, "sprite": 78}]}, {"id": "corner", "bg": 938, "fg": [65, 77, 63, 72]}, {"id": "t_connection", "bg": 938, "fg": [80, 60, 62, 76]}, {"id": "edge", "bg": 938, "fg": [70, 64]}, {"id": "end_piece", "bg": 938, "fg": [57, 79, 69, 67]}, {"bg": 938, "id": "unconnected", "fg": 75}]}, {"id": "t_grass_season_summer", "multitile": true, "fg": 75, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 89}, {"id": "corner", "bg": 937, "fg": [94, 85, 83, 92]}, {"id": "t_connection", "bg": 937, "fg": [87, 96, 90, 82]}, {"id": "edge", "bg": 937, "fg": [84, 88]}, {"id": "end_piece", "bg": 937, "fg": [86, 93, 91, 95]}, {"bg": 937, "id": "unconnected", "fg": 81}]}, {"id": "t_grass_season_autumn", "multitile": true, "fg": 75, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 100}, {"id": "corner", "bg": 946, "fg": [110, 111, 104, 99]}, {"id": "t_connection", "bg": 946, "fg": [101, 112, 106, 103]}, {"id": "edge", "bg": 946, "fg": [105, 108]}, {"id": "end_piece", "bg": 946, "fg": [109, 97, 102, 98]}, {"bg": 946, "id": "unconnected", "fg": 107}]}, {"id": "t_grass_season_winter", "fg": 587}, {"id": "t_door_glass_c", "fg": 113}, {"id": "t_rdoor_c", "fg": 114}, {"id": "t_rdoor_o", "fg": 116}, {"id": "t_rdoor_b", "fg": 115}, {"id": "t_clay", "multitile": true, "fg": 127, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 123}, {"id": "corner", "bg": 938, "fg": [120, 118, 132, 124]}, {"id": "t_connection", "bg": 938, "fg": [125, 121, 128, 117]}, {"id": "edge", "bg": 938, "fg": [122, 129]}, {"id": "end_piece", "bg": 938, "fg": [131, 119, 130, 126]}, {"bg": 938, "id": "unconnected", "fg": 127}]}, {"id": "t_clay_season_summer", "multitile": true, "fg": 127, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 123}, {"id": "corner", "bg": 937, "fg": [120, 118, 132, 124]}, {"id": "t_connection", "bg": 937, "fg": [125, 121, 128, 117]}, {"id": "edge", "bg": 937, "fg": [122, 129]}, {"id": "end_piece", "bg": 937, "fg": [131, 119, 130, 126]}, {"bg": 937, "id": "unconnected", "fg": 127}]}, {"id": "t_clay_season_autumn", "multitile": true, "fg": 127, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 123}, {"id": "corner", "bg": 946, "fg": [120, 118, 132, 124]}, {"id": "t_connection", "bg": 946, "fg": [125, 121, 128, 117]}, {"id": "edge", "bg": 946, "fg": [122, 129]}, {"id": "end_piece", "bg": 946, "fg": [131, 119, 130, 126]}, {"bg": 946, "id": "unconnected", "fg": 127}]}, {"id": "t_clay_season_winter", "fg": 587}, {"id": "t_tatami", "fg": [{"weight": 100, "sprite": 134}, {"weight": 100, "sprite": 133}]}, {"id": "t_bars", "fg": 135}, {"id": "t_fence_season_spring", "multitile": true, "fg": 152, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 143}, {"id": "corner", "bg": 938, "fg": [138, 153, 144, 141]}, {"id": "t_connection", "bg": 938, "fg": [151, 139, 142, 137]}, {"id": "edge", "bg": 938, "fg": [145, 147]}, {"id": "end_piece", "bg": 938, "fg": [136, 148, 150, 149]}, {"bg": 938, "id": "unconnected", "fg": 152}]}, {"id": "t_fence_season_summer", "multitile": true, "fg": 152, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 143}, {"id": "corner", "bg": 937, "fg": [138, 153, 144, 141]}, {"id": "t_connection", "bg": 937, "fg": [151, 139, 142, 137]}, {"id": "edge", "bg": 937, "fg": [145, 147]}, {"id": "end_piece", "bg": 937, "fg": [136, 148, 150, 149]}, {"bg": 937, "id": "unconnected", "fg": 152}]}, {"id": "t_fence_season_autumn", "multitile": true, "fg": 152, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 143}, {"id": "corner", "bg": 946, "fg": [138, 153, 144, 141]}, {"id": "t_connection", "bg": 946, "fg": [151, 139, 142, 137]}, {"id": "edge", "bg": 946, "fg": [145, 147]}, {"id": "end_piece", "bg": 946, "fg": [136, 148, 150, 149]}, {"bg": 946, "id": "unconnected", "fg": 152}]}, {"id": "t_fence_season_winter", "multitile": true, "fg": 152, "bg": 587, "additional_tiles": [{"id": "center", "bg": 587, "fg": 143}, {"id": "corner", "bg": 587, "fg": [138, 153, 144, 141]}, {"id": "t_connection", "bg": 587, "fg": [151, 139, 142, 137]}, {"id": "edge", "bg": 587, "fg": [145, 147]}, {"id": "end_piece", "bg": 587, "fg": [136, 148, 150, 149]}, {"bg": 587, "id": "unconnected", "fg": 152}]}, {"id": "t_fencegate_c_season_spring", "fg": 146, "bg": 938}, {"id": "t_fencegate_c_season_summer", "fg": 146, "bg": 937}, {"id": "t_fencegate_c_season_autumn", "fg": 146, "bg": 946}, {"id": "t_fencegate_c_season_winter", "fg": 146, "bg": 587}, {"id": "t_fencegate_o_season_spring", "fg": 140, "bg": 938}, {"id": "t_fencegate_o_season_summer", "fg": 140, "bg": 937}, {"id": "t_fencegate_o_season_autumn", "fg": 140, "bg": 946}, {"id": "t_fencegate_o_season_winter", "fg": 140, "bg": 587}, {"id": ["t_shrub_hydrangea"], "fg": 155, "bg": 938}, {"id": ["t_shrub_hydrangea_harvested"], "fg": 157, "bg": 938}, {"id": "t_shrub_hydrangea_season_summer", "fg": 156, "bg": 937}, {"id": "t_shrub_hydrangea_season_autumn", "fg": 155, "bg": 946}, {"id": "t_shrub_hydrangea_season_winter", "fg": 154, "bg": 586}, {"id": "t_floor_resin", "multitile": true, "fg": 172, "additional_tiles": [{"id": "center", "fg": 163}, {"id": "corner", "fg": [164, 165, 160, 169]}, {"id": "t_connection", "fg": [161, 167, 159, 171]}, {"id": "edge", "fg": [158, 170]}, {"id": "end_piece", "fg": [166, 162, 173, 168]}, {"id": "unconnected", "fg": 172}]}, {"id": "t_door_glass_o", "fg": 174}, {"id": ["t_shrub_rose"], "fg": 176, "bg": 938}, {"id": "t_shrub_rose_season_summer", "fg": 176, "bg": 937}, {"id": "t_shrub_rose_harvested", "fg": 176, "bg": 946}, {"id": "t_shrub_rose_season_autumn", "fg": 177, "bg": 946}, {"id": "t_shrub_rose_season_winter", "fg": 175, "bg": 586}, {"id": "t_grass_long", "multitile": true, "fg": 191, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 185}, {"id": "corner", "bg": 938, "fg": [182, 178, 181, 180]}, {"id": "t_connection", "bg": 938, "fg": [187, 189, 184, 188]}, {"id": "edge", "bg": 938, "fg": [183, 179]}, {"id": "end_piece", "bg": 938, "fg": [193, 190, 186, 192]}, {"bg": 938, "id": "unconnected", "fg": 191}]}, {"id": "t_grass_long_season_autumn", "multitile": true, "fg": 219, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 213}, {"id": "corner", "bg": 946, "fg": [214, 225, 221, 220]}, {"id": "t_connection", "bg": 946, "fg": [217, 211, 222, 215]}, {"id": "edge", "bg": 946, "fg": [223, 216]}, {"id": "end_piece", "bg": 946, "fg": [210, 212, 218, 224]}, {"bg": 946, "id": "unconnected", "fg": 219}]}, {"id": "t_grass_long_season_summer", "multitile": true, "fg": 194, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 202}, {"id": "corner", "bg": 937, "fg": [206, 209, 199, 207]}, {"id": "t_connection", "bg": 937, "fg": [204, 203, 196, 197]}, {"id": "edge", "bg": 937, "fg": [205, 200]}, {"id": "end_piece", "bg": 937, "fg": [195, 208, 201, 198]}, {"bg": 937, "id": "unconnected", "fg": 194}]}, {"id": "t_grass_long_season_winter", "fg": 586}, {"id": "f_grass_long", "multitile": true, "fg": 191, "additional_tiles": [{"id": "center", "fg": 185}, {"id": "corner", "fg": [182, 178, 181, 180]}, {"id": "t_connection", "fg": [187, 189, 184, 188]}, {"id": "edge", "fg": [183, 179]}, {"id": "end_piece", "fg": [193, 190, 186, 192]}, {"id": "unconnected", "fg": 191}]}, {"id": ["t_linoleum_white", "t_linoleum_white_no_roof"], "multitile": true, "fg": 236, "additional_tiles": [{"id": "center", "fg": 238}, {"id": "corner", "fg": [237, 230, 241, 233]}, {"id": "t_connection", "fg": [227, 229, 228, 226]}, {"id": "edge", "fg": [231, 235]}, {"id": "end_piece", "fg": [232, 234, 240, 239]}, {"id": "unconnected", "fg": 236}]}, {"id": ["t_door_metal_locked", "t_door_metal_pickable"], "fg": 242}, {"id": "t_rock", "fg": 243}, {"id": ["t_stump"], "fg": 244, "bg": 938}, {"id": ["t_stump_season_summer"], "fg": 244, "bg": 937}, {"id": ["t_stump_season_autumn"], "fg": 244, "bg": 946}, {"id": "t_stump_season_winter", "fg": 244, "bg": 586}, {"id": "t_door_lab_c", "fg": 245}, {"id": "t_door_lab_o", "fg": 246}, {"id": "t_ladder_down", "fg": 247}, {"id": "t_wall_g", "multitile": true, "fg": 255, "additional_tiles": [{"id": "center", "fg": 259}, {"id": "corner", "fg": [252, 254, 258, 261]}, {"id": "t_connection", "fg": [253, 248, 256, 263]}, {"id": "edge", "fg": [260, 262]}, {"id": "end_piece", "fg": [249, 257, 250, 251]}, {"id": "unconnected", "fg": 255}]}, {"id": ["t_water_dp", "t_swater_dp"], "multitile": true, "fg": 268, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": [{"weight": 1, "sprite": 272}, {"weight": 1, "sprite": 269}, {"weight": 1, "sprite": 271}]}, {"id": "corner", "bg": 938, "fg": [276, 278, 277, 266]}, {"id": "t_connection", "bg": 938, "fg": [270, 267, 273, 274]}, {"id": "edge", "bg": 938, "fg": [280, 265]}, {"id": "end_piece", "bg": 938, "fg": [281, 275, 279, 264]}, {"bg": 938, "id": "unconnected", "fg": 268}]}, {"id": ["t_water_dp_season_summer", "t_swater_dp_season_summer"], "multitile": true, "fg": 268, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": [{"weight": 1, "sprite": 272}, {"weight": 1, "sprite": 269}, {"weight": 1, "sprite": 271}]}, {"id": "corner", "bg": 937, "fg": [276, 278, 277, 266]}, {"id": "t_connection", "bg": 937, "fg": [270, 267, 273, 274]}, {"id": "edge", "bg": 937, "fg": [280, 265]}, {"id": "end_piece", "bg": 937, "fg": [281, 275, 279, 264]}, {"bg": 937, "id": "unconnected", "fg": 268}]}, {"id": ["t_water_dp_season_autumn", "t_swater_dp_season_autumn"], "multitile": true, "fg": 268, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": [{"weight": 1, "sprite": 272}, {"weight": 1, "sprite": 269}, {"weight": 1, "sprite": 271}]}, {"id": "corner", "bg": 946, "fg": [276, 278, 277, 266]}, {"id": "t_connection", "bg": 946, "fg": [270, 267, 273, 274]}, {"id": "edge", "bg": 946, "fg": [280, 265]}, {"id": "end_piece", "bg": 946, "fg": [281, 275, 279, 264]}, {"bg": 946, "id": "unconnected", "fg": 268}]}, {"id": ["t_water_dp_season_winter", "t_swater_dp_season_winter"], "multitile": true, "fg": 268, "bg": 587, "additional_tiles": [{"id": "center", "bg": 587, "fg": [{"weight": 1, "sprite": 272}, {"weight": 1, "sprite": 269}, {"weight": 1, "sprite": 271}]}, {"id": "corner", "bg": 587, "fg": [276, 278, 277, 266]}, {"id": "t_connection", "bg": 587, "fg": [270, 267, 273, 274]}, {"id": "edge", "bg": 587, "fg": [280, 265]}, {"id": "end_piece", "bg": 587, "fg": [281, 275, 279, 264]}, {"bg": 587, "id": "unconnected", "fg": 268}]}, {"id": "t_floor", "multitile": true, "fg": 292, "additional_tiles": [{"id": "center", "fg": 282}, {"id": "corner", "fg": [283, 294, 289, 286]}, {"id": "t_connection", "fg": [297, 287, 284, 285]}, {"id": "edge", "fg": [293, 295]}, {"id": "end_piece", "fg": [291, 296, 290, 288]}, {"id": "unconnected", "fg": 292}]}, {"id": "t_concrete", "multitile": true, "fg": 306, "additional_tiles": [{"id": "center", "fg": 301}, {"id": "corner", "fg": [304, 300, 299, 311]}, {"id": "t_connection", "fg": [307, 303, 313, 298]}, {"id": "edge", "fg": [308, 310]}, {"id": "end_piece", "fg": [312, 309, 305, 302]}, {"id": "unconnected", "fg": 306}]}, {"id": "t_concrete_season_winter", "fg": 587}, {"id": "t_door_metal_c_peep", "fg": 314}, {"id": "t_strconc_floor", "multitile": true, "fg": 324, "additional_tiles": [{"id": "center", "fg": 326}, {"id": "corner", "fg": [318, 321, 325, 329]}, {"id": "t_connection", "fg": [316, 327, 315, 317]}, {"id": "edge", "fg": [320, 322]}, {"id": "end_piece", "fg": [323, 330, 319, 328]}, {"id": "unconnected", "fg": 324}]}, {"id": ["t_window", "t_window_alarm"], "fg": 333}, {"id": "t_window_empty", "fg": 335}, {"id": "t_window_domestic", "fg": 331}, {"id": "t_window_open", "fg": 332}, {"id": "t_window_no_curtains", "fg": 336}, {"id": "t_window_no_curtains_open", "fg": 334}, {"id": ["t_linoleum_gray", "t_linoluem_gray_no_roof"], "multitile": true, "fg": 338, "additional_tiles": [{"id": "center", "fg": 339}, {"id": "corner", "fg": [351, 349, 343, 337]}, {"id": "t_connection", "fg": [346, 344, 342, 340]}, {"id": "edge", "fg": [348, 352]}, {"id": "end_piece", "fg": [350, 345, 347, 341]}, {"id": "unconnected", "fg": 338}]}, {"id": "t_thconc_floor", "multitile": true, "fg": 355, "additional_tiles": [{"id": "center", "fg": 353}, {"id": "corner", "fg": [364, 354, 362, 367]}, {"id": "t_connection", "fg": [359, 365, 368, 357]}, {"id": "edge", "fg": [361, 363]}, {"id": "end_piece", "fg": [366, 356, 358, 360]}, {"id": "unconnected", "fg": 355}]}, {"id": "f_grass_tall", "multitile": true, "fg": 374, "additional_tiles": [{"id": "center", "fg": 371}, {"id": "corner", "fg": [381, 384, 378, 369]}, {"id": "t_connection", "fg": [376, 379, 372, 382]}, {"id": "edge", "fg": [370, 375]}, {"id": "end_piece", "fg": [380, 373, 383, 377]}, {"id": "unconnected", "fg": 374}]}, {"id": "f_grass_tall_summer", "multitile": true, "fg": 389, "additional_tiles": [{"id": "center", "fg": 398}, {"id": "corner", "fg": [397, 391, 390, 388]}, {"id": "t_connection", "fg": [385, 396, 392, 394]}, {"id": "edge", "fg": [399, 387]}, {"id": "end_piece", "fg": [400, 395, 386, 393]}, {"id": "unconnected", "fg": 389}]}, {"id": "f_grass_tall_autumn", "multitile": true, "fg": 405, "additional_tiles": [{"id": "center", "fg": 414}, {"id": "corner", "fg": [402, 408, 404, 409]}, {"id": "t_connection", "fg": [406, 415, 407, 401]}, {"id": "edge", "fg": [412, 403]}, {"id": "end_piece", "fg": [416, 411, 410, 413]}, {"id": "unconnected", "fg": 405}]}, {"id": "t_grass_tall", "multitile": true, "fg": 374, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 371}, {"id": "corner", "bg": 938, "fg": [381, 384, 378, 369]}, {"id": "t_connection", "bg": 938, "fg": [376, 379, 372, 382]}, {"id": "edge", "bg": 938, "fg": [370, 375]}, {"id": "end_piece", "bg": 938, "fg": [380, 373, 383, 377]}, {"bg": 938, "id": "unconnected", "fg": 374}]}, {"id": "t_grass_tall_season_summer", "multitile": true, "fg": 389, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 398}, {"id": "corner", "bg": 937, "fg": [397, 391, 390, 388]}, {"id": "t_connection", "bg": 937, "fg": [385, 396, 392, 394]}, {"id": "edge", "bg": 937, "fg": [399, 387]}, {"id": "end_piece", "bg": 937, "fg": [400, 395, 386, 393]}, {"bg": 937, "id": "unconnected", "fg": 389}]}, {"id": "t_grass_tall_season_autumn", "multitile": true, "fg": 405, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 414}, {"id": "corner", "bg": 946, "fg": [402, 408, 404, 409]}, {"id": "t_connection", "bg": 946, "fg": [406, 415, 407, 401]}, {"id": "edge", "bg": 946, "fg": [412, 403]}, {"id": "end_piece", "bg": 946, "fg": [416, 411, 410, 413]}, {"bg": 946, "id": "unconnected", "fg": 405}]}, {"id": "t_grass_tall_season_winter", "fg": 586}, {"id": "t_pavement", "fg": 417}, {"id": "t_pavement_season_winter", "fg": 587}, {"id": "t_pit", "fg": 418, "bg": [{"weight": 100, "sprite": 938}, {"weight": 100, "sprite": 943}]}, {"id": "t_pit_season_summer", "fg": 418, "bg": [{"weight": 100, "sprite": 937}, {"weight": 100, "sprite": 941}]}, {"id": "t_pit_season_autumn", "fg": 418, "bg": [{"weight": 100, "sprite": 946}, {"weight": 100, "sprite": 948}]}, {"id": ["t_sidewalk", "t_sidewalk_bg_dp"], "multitile": true, "fg": 429, "additional_tiles": [{"id": "center", "fg": 428}, {"id": "corner", "fg": [432, 420, 419, 424]}, {"id": "t_connection", "fg": [423, 430, 422, 431]}, {"id": "edge", "fg": [433, 426]}, {"id": "end_piece", "fg": [427, 421, 434, 425]}, {"id": "unconnected", "fg": 429}]}, {"id": "t_sidewalk_season_winter", "fg": 587}, {"id": "t_moss", "fg": 435}, {"id": "t_moss_season_winter", "fg": 586}, {"id": "t_wall_p", "multitile": true, "fg": 447, "additional_tiles": [{"id": "center", "fg": 451}, {"id": "corner", "fg": [440, 442, 444, 449]}, {"id": "t_connection", "fg": [439, 438, 437, 436]}, {"id": "edge", "fg": [443, 446]}, {"id": "end_piece", "fg": [450, 445, 441, 448]}, {"id": "unconnected", "fg": 447}]}, {"id": "t_fence_barbed_season_spring", "multitile": true, "fg": 463, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 453}, {"id": "corner", "bg": 938, "fg": [454, 458, 456, 457]}, {"id": "t_connection", "bg": 938, "fg": [464, 459, 462, 467]}, {"id": "edge", "bg": 938, "fg": [455, 452]}, {"id": "end_piece", "bg": 938, "fg": [466, 460, 461, 465]}, {"bg": 938, "id": "unconnected", "fg": 463}]}, {"id": "t_fence_barbed_season_summer", "multitile": true, "fg": 463, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 453}, {"id": "corner", "bg": 937, "fg": [454, 458, 456, 457]}, {"id": "t_connection", "bg": 937, "fg": [464, 459, 462, 467]}, {"id": "edge", "bg": 937, "fg": [455, 452]}, {"id": "end_piece", "bg": 937, "fg": [466, 460, 461, 465]}, {"bg": 937, "id": "unconnected", "fg": 463}]}, {"id": "t_fence_barbed_season_autumn", "multitile": true, "fg": 463, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 453}, {"id": "corner", "bg": 946, "fg": [454, 458, 456, 457]}, {"id": "t_connection", "bg": 946, "fg": [464, 459, 462, 467]}, {"id": "edge", "bg": 946, "fg": [455, 452]}, {"id": "end_piece", "bg": 946, "fg": [466, 460, 461, 465]}, {"bg": 946, "id": "unconnected", "fg": 463}]}, {"id": "t_fence_barbed_season_winter", "multitile": true, "fg": 463, "bg": 587, "additional_tiles": [{"id": "center", "bg": 587, "fg": 453}, {"id": "corner", "bg": 587, "fg": [454, 458, 456, 457]}, {"id": "t_connection", "bg": 587, "fg": [464, 459, 462, 467]}, {"id": "edge", "bg": 587, "fg": [455, 452]}, {"id": "end_piece", "bg": 587, "fg": [466, 460, 461, 465]}, {"bg": 587, "id": "unconnected", "fg": 463}]}, {"id": "t_wall_wood", "multitile": true, "fg": 476, "additional_tiles": [{"id": "center", "fg": 479}, {"id": "corner", "fg": [482, 483, 471, 473]}, {"id": "t_connection", "fg": [474, 475, 470, 477]}, {"id": "edge", "fg": [468, 481]}, {"id": "end_piece", "fg": [469, 472, 478, 480]}, {"id": "unconnected", "fg": 476}]}, {"id": ["t_water_sh", "t_swater_sh"], "multitile": true, "fg": 501, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": [{"weight": 1, "sprite": 496}, {"weight": 1, "sprite": 491}, {"weight": 1, "sprite": 485}]}, {"id": "corner", "bg": 938, "fg": [487, 490, 488, 499]}, {"id": "t_connection", "bg": 938, "fg": [494, 495, 498, 497]}, {"id": "edge", "bg": 938, "fg": [486, 484]}, {"id": "end_piece", "bg": 938, "fg": [493, 500, 489, 492]}, {"bg": 938, "id": "unconnected", "fg": 501}]}, {"id": ["t_water_sh_season_summer", "t_swater_sh_season_summer"], "multitile": true, "fg": 501, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": [{"weight": 1, "sprite": 496}, {"weight": 1, "sprite": 491}, {"weight": 1, "sprite": 485}]}, {"id": "corner", "bg": 937, "fg": [487, 490, 488, 499]}, {"id": "t_connection", "bg": 937, "fg": [494, 495, 498, 497]}, {"id": "edge", "bg": 937, "fg": [486, 484]}, {"id": "end_piece", "bg": 937, "fg": [493, 500, 489, 492]}, {"bg": 937, "id": "unconnected", "fg": 501}]}, {"id": ["t_water_sh_season_autumn", "t_swater_sh_season_autumn"], "multitile": true, "fg": 501, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": [{"weight": 1, "sprite": 496}, {"weight": 1, "sprite": 491}, {"weight": 1, "sprite": 485}]}, {"id": "corner", "bg": 946, "fg": [487, 490, 488, 499]}, {"id": "t_connection", "bg": 946, "fg": [494, 495, 498, 497]}, {"id": "edge", "bg": 946, "fg": [486, 484]}, {"id": "end_piece", "bg": 946, "fg": [493, 500, 489, 492]}, {"bg": 946, "id": "unconnected", "fg": 501}]}, {"id": ["t_water_sh_season_winter", "t_swater_sh_season_winter"], "multitile": true, "fg": 501, "bg": 587, "additional_tiles": [{"id": "center", "bg": 587, "fg": [{"weight": 1, "sprite": 496}, {"weight": 1, "sprite": 491}, {"weight": 1, "sprite": 485}]}, {"id": "corner", "bg": 587, "fg": [487, 490, 488, 499]}, {"id": "t_connection", "bg": 587, "fg": [494, 495, 498, 497]}, {"id": "edge", "bg": 587, "fg": [486, 484]}, {"id": "end_piece", "bg": 587, "fg": [493, 500, 489, 492]}, {"bg": 587, "id": "unconnected", "fg": 501}]}, {"id": ["t_shrub_lilac"], "fg": 505, "bg": 938}, {"id": ["t_shrub_lilac_harvested"], "fg": 502, "bg": []}, {"id": "t_shrub_lilac_season_summer", "fg": 503, "bg": 937}, {"id": "t_shrub_lilac_season_autumn", "fg": 502, "bg": 946}, {"id": "t_shrub_lilac_season_winter", "fg": 504, "bg": 586}, {"id": "t_rock_wall", "multitile": true, "fg": 508, "additional_tiles": [{"id": "center", "fg": 506}, {"id": "corner", "fg": [515, 521, 513, 512]}, {"id": "t_connection", "fg": [514, 509, 520, 519]}, {"id": "edge", "fg": [507, 510]}, {"id": "end_piece", "fg": [511, 518, 516, 517]}, {"id": "unconnected", "fg": 508}]}, {"id": "t_reinforced_glass", "multitile": true, "fg": 532, "additional_tiles": [{"id": "center", "fg": 534}, {"id": "corner", "fg": [528, 537, 522, 531]}, {"id": "t_connection", "fg": [526, 536, 527, 529]}, {"id": "edge", "fg": [524, 523]}, {"id": "end_piece", "fg": [530, 525, 535, 533]}, {"id": "unconnected", "fg": 532}]}, {"id": "t_carpet_purple", "multitile": true, "fg": 548, "additional_tiles": [{"id": "center", "fg": 538}, {"id": "corner", "fg": [552, 542, 547, 539]}, {"id": "t_connection", "fg": [553, 540, 551, 544]}, {"id": "edge", "fg": [541, 545]}, {"id": "end_piece", "fg": [546, 550, 549, 543]}, {"id": "unconnected", "fg": 548}]}, {"id": "t_grass_dead", "multitile": true, "fg": 562, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 568}, {"id": "corner", "bg": 938, "fg": [556, 567, 554, 569]}, {"id": "t_connection", "bg": 938, "fg": [560, 563, 565, 559]}, {"id": "edge", "bg": 938, "fg": [566, 557]}, {"id": "end_piece", "bg": 938, "fg": [564, 558, 561, 555]}, {"bg": 938, "id": "unconnected", "fg": 562}]}, {"id": "t_grass_dead_season_summer", "multitile": true, "fg": 562, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 568}, {"id": "corner", "bg": 937, "fg": [556, 567, 554, 569]}, {"id": "t_connection", "bg": 937, "fg": [560, 563, 565, 559]}, {"id": "edge", "bg": 937, "fg": [566, 557]}, {"id": "end_piece", "bg": 937, "fg": [564, 558, 561, 555]}, {"bg": 937, "id": "unconnected", "fg": 562}]}, {"id": "t_grass_dead_season_autumn", "multitile": true, "fg": 562, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 568}, {"id": "corner", "bg": 946, "fg": [556, 567, 554, 569]}, {"id": "t_connection", "bg": 946, "fg": [560, 563, 565, 559]}, {"id": "edge", "bg": 946, "fg": [566, 557]}, {"id": "end_piece", "bg": 946, "fg": [564, 558, 561, 555]}, {"bg": 946, "id": "unconnected", "fg": 562}]}, {"id": "t_grass_dead_season_winter", "fg": 587}, {"id": "t_brick_wall", "multitile": true, "fg": 576, "additional_tiles": [{"id": "center", "fg": 572}, {"id": "corner", "fg": [580, 577, 584, 573]}, {"id": "t_connection", "fg": [583, 585, 579, 582]}, {"id": "edge", "fg": [574, 571]}, {"id": "end_piece", "fg": [575, 581, 570, 578]}, {"id": "unconnected", "fg": 576}]}, {"id": "t_floor_wax", "multitile": true, "fg": 592, "additional_tiles": [{"id": "center", "fg": 597}, {"id": "corner", "fg": [598, 595, 600, 593]}, {"id": "t_connection", "fg": [594, 596, 602, 603]}, {"id": "edge", "fg": [588, 591]}, {"id": "end_piece", "fg": [599, 590, 589, 601]}, {"id": "unconnected", "fg": 592}]}, {"id": "t_wall_resin", "multitile": true, "fg": 609, "additional_tiles": [{"id": "center", "fg": 612}, {"id": "corner", "fg": [614, 620, 615, 606]}, {"id": "t_connection", "fg": [616, 607, 617, 608]}, {"id": "edge", "fg": [621, 618]}, {"id": "end_piece", "fg": [611, 604, 613, 619]}, {"id": "unconnected", "fg": 609}]}, {"id": "t_resin_hole_c", "fg": 605}, {"id": "t_resin_hole_o", "fg": 610}, {"id": "t_wall_glass", "multitile": true, "fg": 631, "additional_tiles": [{"id": "center", "fg": 628}, {"id": "corner", "fg": [623, 629, 626, 624]}, {"id": "t_connection", "fg": [639, 622, 635, 634]}, {"id": "edge", "fg": [630, 632]}, {"id": "end_piece", "fg": [640, 625, 641, 627]}, {"id": "unconnected", "fg": 631}]}, {"id": "t_woodchips", "fg": 642}, {"id": "t_woodchips_season_winter", "fg": 587}, {"id": "t_fungus", "multitile": true, "fg": 648, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 658}, {"id": "corner", "bg": 938, "fg": [643, 651, 656, 653]}, {"id": "t_connection", "bg": 938, "fg": [645, 657, 650, 646]}, {"id": "edge", "bg": 938, "fg": [655, 652]}, {"id": "end_piece", "bg": 938, "fg": [647, 654, 649, 644]}, {"bg": 938, "id": "unconnected", "fg": 648}]}, {"id": "t_fungus_season_summer", "multitile": true, "fg": 648, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 658}, {"id": "corner", "bg": 937, "fg": [643, 651, 656, 653]}, {"id": "t_connection", "bg": 937, "fg": [645, 657, 650, 646]}, {"id": "edge", "bg": 937, "fg": [655, 652]}, {"id": "end_piece", "bg": 937, "fg": [647, 654, 649, 644]}, {"bg": 937, "id": "unconnected", "fg": 648}]}, {"id": "t_fungus_season_autumn", "multitile": true, "fg": 648, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 658}, {"id": "corner", "bg": 946, "fg": [643, 651, 656, 653]}, {"id": "t_connection", "bg": 946, "fg": [645, 657, 650, 646]}, {"id": "edge", "bg": 946, "fg": [655, 652]}, {"id": "end_piece", "bg": 946, "fg": [647, 654, 649, 644]}, {"bg": 946, "id": "unconnected", "fg": 648}]}, {"id": "t_fungus_season_winter", "multitile": true, "fg": 648, "bg": 587, "additional_tiles": [{"id": "center", "bg": 587, "fg": 658}, {"id": "corner", "bg": 587, "fg": [643, 651, 656, 653]}, {"id": "t_connection", "bg": 587, "fg": [645, 657, 650, 646]}, {"id": "edge", "bg": 587, "fg": [655, 652]}, {"id": "end_piece", "bg": 587, "fg": [647, 654, 649, 644]}, {"bg": 587, "id": "unconnected", "fg": 648}]}, {"id": "t_metal_floor", "multitile": true, "fg": 666, "additional_tiles": [{"id": "center", "fg": 661}, {"id": "corner", "fg": [669, 660, 670, 671]}, {"id": "t_connection", "fg": [659, 668, 662, 664]}, {"id": "edge", "fg": [673, 665]}, {"id": "end_piece", "fg": [674, 672, 667, 663]}, {"id": "unconnected", "fg": 666}]}, {"id": "t_door_boarded", "fg": 675}, {"id": "t_door_boarded_damaged", "fg": 676}, {"id": "t_fence_wire", "multitile": true, "fg": 680, "additional_tiles": [{"id": "center", "fg": 692}, {"id": "corner", "fg": [690, 691, 681, 687]}, {"id": "t_connection", "fg": [685, 682, 678, 689]}, {"id": "edge", "fg": [684, 686]}, {"id": "end_piece", "fg": [683, 677, 688, 679]}, {"id": "unconnected", "fg": 680}]}, {"id": "t_railroad_rubble", "fg": 693}, {"id": "t_gravel", "fg": 693}, {"id": "t_railroad_rubble_season_winter", "fg": 587}, {"id": "t_sand", "fg": 694}, {"id": "t_sand_season_winter", "fg": 587}, {"id": "f_floor_scrap", "multitile": true, "fg": 703, "additional_tiles": [{"id": "center", "fg": 701}, {"id": "corner", "fg": [697, 700, 696, 699]}, {"id": "t_connection", "fg": [707, 704, 695, 705]}, {"id": "edge", "fg": [702, 709]}, {"id": "end_piece", "fg": [698, 708, 706, 710]}, {"id": "unconnected", "fg": 703}]}, {"id": ["t_water_moving_sh", "t_swater_moving_sh"], "multitile": true, "fg": 724, "bg": 938, "additional_tiles": [{"id": "center", "bg": 938, "fg": 714}, {"id": "corner", "bg": 938, "fg": [717, 718, 715, 725]}, {"id": "t_connection", "bg": 938, "fg": [723, 720, 713, 716]}, {"id": "edge", "bg": 938, "fg": [719, 726]}, {"id": "end_piece", "bg": 938, "fg": [712, 721, 722, 711]}, {"bg": 938, "id": "unconnected", "fg": 724}]}, {"id": ["t_water_moving_sh_season_summer", "t_swater_moving_sh_season_summer"], "multitile": true, "fg": 724, "bg": 937, "additional_tiles": [{"id": "center", "bg": 937, "fg": 714}, {"id": "corner", "bg": 937, "fg": [717, 718, 715, 725]}, {"id": "t_connection", "bg": 937, "fg": [723, 720, 713, 716]}, {"id": "edge", "bg": 937, "fg": [719, 726]}, {"id": "end_piece", "bg": 937, "fg": [712, 721, 722, 711]}, {"bg": 937, "id": "unconnected", "fg": 724}]}, {"id": ["t_water_moving_sh_season_autumn", "t_swater_moving_sh_season_autumn"], "multitile": true, "fg": 724, "bg": 946, "additional_tiles": [{"id": "center", "bg": 946, "fg": 714}, {"id": "corner", "bg": 946, "fg": [717, 718, 715, 725]}, {"id": "t_connection", "bg": 946, "fg": [723, 720, 713, 716]}, {"id": "edge", "bg": 946, "fg": [719, 726]}, {"id": "end_piece", "bg": 946, "fg": [712, 721, 722, 711]}, {"bg": 946, "id": "unconnected", "fg": 724}]}, {"id": ["t_water_moving_sh_season_winter", "t_swater_moving_sh_season_winter"], "multitile": true, "fg": 724, "bg": 587, "additional_tiles": [{"id": "center", "bg": 587, "fg": 714}, {"id": "corner", "bg": 587, "fg": [717, 718, 715, 725]}, {"id": "t_connection", "bg": 587, "fg": [723, 720, 713, 716]}, {"id": "edge", "bg": 587, "fg": [719, 726]}, {"id": "end_piece", "bg": 587, "fg": [712, 721, 722, 711]}, {"bg": 587, "id": "unconnected", "fg": 724}]}, {"id": "t_elevator", "fg": [{"weight": 100, "sprite": 727}, {"weight": 100, "sprite": 728}]}, {"id": "t_rock_floor", "multitile": true, "fg": 738, "additional_tiles": [{"id": "center", "fg": 735}, {"id": "corner", "fg": [741, 730, 739, 734]}, {"id": "t_connection", "fg": [740, 736, 732, 729]}, {"id": "edge", "fg": [731, 742]}, {"id": "end_piece", "fg": [744, 737, 743, 733]}, {"id": "unconnected", "fg": 738}]}, {"id": "t_carpet_yellow", "multitile": true, "fg": 750, "additional_tiles": [{"id": "center", "fg": 746}, {"id": "corner", "fg": [749, 756, 754, 745]}, {"id": "t_connection", "fg": [759, 748, 747, 758]}, {"id": "edge", "fg": [755, 753]}, {"id": "end_piece", "fg": [752, 760, 757, 751]}, {"id": "unconnected", "fg": 750}]}, {"id": "t_stairs_down", "fg": 762}, {"id": "t_wood_stairs_down", "fg": 761}, {"id": "t_door_c", "fg": 766}, {"id": "t_door_locked", "fg": 764}, {"id": "t_door_o", "fg": 763}, {"id": "t_door_b", "fg": 765}, {"id": ["t_shrub_raspberry"], "fg": 770, "bg": 938}, {"id": ["t_shrub_raspberry_harvested"], "fg": 767, "bg": 937}, {"id": "t_shrub_raspberry_season_summer", "fg": 768, "bg": 937}, {"id": "t_shrub_raspberry_season_autumn", "fg": 770, "bg": 946}, {"id": "t_shrub_raspberry_season_winter", "fg": 769, "bg": 586}, {"id": "t_pit_shallow", "fg": 771, "bg": [{"weight": 100, "sprite": 938}, {"weight": 100, "sprite": 943}]}, {"id": "t_pit_shallow_season_summer", "fg": 771, "bg": [{"weight": 100, "sprite": 937}, {"weight": 100, "sprite": 941}]}, {"id": "t_pit_shallow_season_autumn", "fg": 771, "bg": [{"weight": 100, "sprite": 946}, {"weight": 100, "sprite": 948}]}, {"id": "t_wall_r", "multitile": true, "fg": 783, "additional_tiles": [{"id": "center", "fg": 787}, {"id": "corner", "fg": [772, 774, 780, 775]}, {"id": "t_connection", "fg": [781, 778, 776, 785]}, {"id": "edge", "fg": [773, 777]}, {"id": "end_piece", "fg": [784, 779, 786, 782]}, {"id": "unconnected", "fg": 783}]}, {"id": ["t_water_pool", "t_water_pool_shallow"], "multitile": true, "fg": 790, "additional_tiles": [{"id": "center", "fg": [{"weight": 1, "sprite": 496}, {"weight": 1, "sprite": 491}, {"weight": 1, "sprite": 485}]}, {"id": "corner", "fg": [796, 795, 799, 794]}, {"id": "t_connection", "fg": [792, 788, 800, 789]}, {"id": "edge", "fg": [798, 791]}, {"id": "end_piece", "fg": [793, 802, 797, 801]}, {"id": "unconnected", "fg": 790}]}, {"id": "t_dirtfloor", "multitile": true, "fg": 815, "additional_tiles": [{"id": "center", "fg": 818}, {"id": "corner", "fg": [811, 808, 806, 813]}, {"id": "t_connection", "fg": [805, 803, 810, 817]}, {"id": "edge", "fg": [814, 804]}, {"id": "end_piece", "fg": [809, 816, 812, 807]}, {"id": "unconnected", "fg": 815}]}, {"id": "t_concrete_wall", "multitile": true, "fg": 825, "additional_tiles": [{"id": "center", "fg": 831}, {"id": "corner", "fg": [829, 832, 824, 821]}, {"id": "t_connection", "fg": [827, 823, 834, 828]}, {"id": "edge", "fg": [819, 826]}, {"id": "end_piece", "fg": [822, 833, 830, 820]}, {"id": "unconnected", "fg": 825}]}, {"id": "f_scrap_wall", "multitile": true, "fg": 836, "additional_tiles": [{"id": "center", "fg": 845}, {"id": "corner", "fg": [843, 850, 839, 837]}, {"id": "t_connection", "fg": [838, 851, 835, 841]}, {"id": "edge", "fg": [840, 849]}, {"id": "end_piece", "fg": [847, 848, 842, 844]}, {"id": "unconnected", "fg": 836}]}, {"id": "t_carpet_green", "multitile": true, "fg": 863, "additional_tiles": [{"id": "center", "fg": 855}, {"id": "corner", "fg": [852, 862, 853, 867]}, {"id": "t_connection", "fg": [859, 854, 866, 865]}, {"id": "edge", "fg": [856, 861]}, {"id": "end_piece", "fg": [858, 860, 864, 857]}, {"id": "unconnected", "fg": 863}]}, {"id": "t_wall_b", "multitile": true, "fg": 876, "additional_tiles": [{"id": "center", "fg": 883}, {"id": "corner", "fg": [869, 871, 874, 880]}, {"id": "t_connection", "fg": [882, 872, 875, 878]}, {"id": "edge", "fg": [877, 879]}, {"id": "end_piece", "fg": [870, 881, 873, 868]}, {"id": "unconnected", "fg": 876}]}, {"id": ["t_shrub_grape"], "fg": 884, "bg": 938}, {"id": ["t_shrub_grape_harvested"], "fg": 887, "bg": 937}, {"id": "t_shrub_grape_season_summer", "fg": 886, "bg": 937}, {"id": "t_shrub_grape_season_autumn", "fg": 884, "bg": 946}, {"id": "t_shrub_grape_season_winter", "fg": 885, "bg": 586}, {"id": "t_floor_waxed", "multitile": true, "fg": 894, "additional_tiles": [{"id": "center", "fg": 901}, {"id": "corner", "fg": [896, 889, 898, 888]}, {"id": "t_connection", "fg": [902, 899, 890, 897]}, {"id": "edge", "fg": [895, 892]}, {"id": "end_piece", "fg": [903, 893, 891, 900]}, {"id": "unconnected", "fg": 894}]}, {"id": "t_wall_log", "multitile": true, "fg": 918, "bg": 938, "additional_tiles": [{"id": "center", "fg": 913}, {"id": "corner", "fg": [910, 919, 911, 917]}, {"id": "t_connection", "fg": [905, 914, 915, 909]}, {"id": "edge", "bg": 938, "fg": [916, 908]}, {"id": "end_piece", "bg": 938, "fg": [904, 907, 912, 906]}, {"bg": 938, "id": "unconnected", "fg": 918}]}, {"id": "t_wall_y", "multitile": true, "fg": 930, "additional_tiles": [{"id": "center", "fg": 925}, {"id": "corner", "fg": [924, 932, 934, 935]}, {"id": "t_connection", "fg": [926, 927, 920, 933]}, {"id": "edge", "fg": [923, 928]}, {"id": "end_piece", "fg": [929, 922, 921, 931]}, {"id": "unconnected", "fg": 930}]}, {"id": "t_dirt", "fg": [{"weight": 100, "sprite": 938}, {"weight": 70, "sprite": 943}, {"weight": 30, "sprite": 947}, {"weight": 30, "sprite": 940}, {"weight": 10, "sprite": 936}, {"weight": 20, "sprite": 939}, {"weight": 20, "sprite": 942}, {"weight": 10, "sprite": 945}, {"weight": 10, "sprite": 944}], "bg": [{"weight": 100, "sprite": 938}, {"weight": 200, "sprite": 943}]}, {"id": "t_dirt_season_summer", "fg": [{"weight": 100, "sprite": 937}, {"weight": 100, "sprite": 941}, {"weight": 30, "sprite": 947}, {"weight": 30, "sprite": 940}, {"weight": 10, "sprite": 936}, {"weight": 12, "sprite": 939}, {"weight": 12, "sprite": 942}, {"weight": 7, "sprite": 945}, {"weight": 6, "sprite": 944}], "bg": [{"weight": 100, "sprite": 937}, {"weight": 100, "sprite": 941}]}, {"id": "t_dirt_season_autumn", "fg": [{"weight": 100, "sprite": 946}, {"weight": 100, "sprite": 948}, {"weight": 30, "sprite": 947}, {"weight": 30, "sprite": 940}, {"weight": 10, "sprite": 936}, {"weight": 6, "sprite": 939}, {"weight": 6, "sprite": 942}, {"weight": 6, "sprite": 945}, {"weight": 4, "sprite": 944}], "bg": [{"weight": 100, "sprite": 946}, {"weight": 100, "sprite": 948}]}, {"id": "t_dirt_season_winter", "fg": 587}, {"id": "t_wall_w", "multitile": true, "fg": 959, "additional_tiles": [{"id": "center", "fg": 955}, {"id": "corner", "fg": [951, 954, 963, 949]}, {"id": "t_connection", "fg": [956, 958, 950, 961]}, {"id": "edge", "fg": [964, 953]}, {"id": "end_piece", "fg": [952, 957, 960, 962]}, {"id": "unconnected", "fg": 959}]}, {"id": "vp_door_trunk", "fg": 966, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 965}]}, {"id": ["vp_headlight", "vp_headlight_reinforced"], "fg": 967}, {"id": "vp_halfboard_horizontal_2", "fg": 969, "rotates": true}, {"id": "vp_halfboard_sw", "fg": 971, "rotates": true}, {"id": "vp_halfboard_vertical_2", "fg": 974, "rotates": true}, {"id": "vp_halfboard_cover", "fg": 976, "rotates": true}, {"id": "vp_halfboard_se", "fg": 970, "rotates": true}, {"id": "vp_halfboard_nw", "fg": 975, "rotates": true}, {"id": "vp_halfboard_ne", "fg": 968, "rotates": true}, {"id": "vp_halfboard_horizontal", "fg": 973, "rotates": true}, {"id": "vp_halfboard_vertical", "fg": 972, "rotates": true}, {"id": "vp_door", "fg": 977, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 978}]}, {"id": "vp_windshield", "fg": 979, "rotates": true, "multitile": true, "additional_tiles": [{"id": "center", "fg": [981, 983, 980, 982]}, {"id": "edge", "fg": 979}, {"id": "unconnected", "fg": 979}, {"id": "end_piece", "fg": [981, 983, 980, 982]}, {"id": "t_connection", "fg": [981, 983, 980, 982]}, {"id": "corner", "fg": [981, 983, 980, 982]}]}, {"id": "f_boulder_medium", "fg": 984}, {"id": "f_table", "multitile": true, "fg": 985, "additional_tiles": [{"id": "center", "fg": 993}, {"id": "corner", "fg": [996, 990, 999, 991]}, {"id": "t_connection", "fg": [988, 997, 986, 992]}, {"id": "edge", "fg": [995, 989]}, {"id": "end_piece", "fg": [998, 994, 987, 1000]}, {"id": "unconnected", "fg": 985}]}, {"id": "f_counter", "multitile": true, "fg": 1009, "additional_tiles": [{"id": "center", "fg": 1003}, {"id": "corner", "fg": [1005, 1016, 1014, 1011]}, {"id": "t_connection", "fg": [1015, 1002, 1001, 1007]}, {"id": "edge", "fg": [1013, 1006]}, {"id": "end_piece", "fg": [1008, 1004, 1010, 1012]}, {"id": "unconnected", "fg": 1009}]}, {"id": "f_rack_wood", "fg": 1017}, {"id": "f_sign", "fg": 1018}, {"id": "f_rubble", "fg": 1019}, {"id": "f_rubble_rock", "fg": 1020}, {"id": "f_boulder_small", "fg": 1021}, {"id": "f_cupboard", "multitile": true, "fg": 1022, "additional_tiles": [{"id": "center", "fg": 1036}, {"id": "corner", "fg": [1037, 1025, 1033, 1028]}, {"id": "t_connection", "fg": [1031, 1034, 1030, 1027]}, {"id": "edge", "fg": [1023, 1032]}, {"id": "end_piece", "fg": [1026, 1029, 1035, 1024]}, {"id": "unconnected", "fg": 1022}]}, {"id": ["f_displaycase"], "fg": 1038}, {"id": "f_wreckage", "fg": 1039}, {"id": "f_recycle_bin", "fg": 1040}, {"id": "f_bathtub", "multitile": true, "fg": 1054, "additional_tiles": [{"id": "center", "fg": 1056}, {"id": "corner", "fg": [1055, 1050, 1052, 1044]}, {"id": "t_connection", "fg": [1051, 1053, 1045, 1047]}, {"id": "edge", "fg": [1041, 1043]}, {"id": "end_piece", "fg": [1042, 1049, 1046, 1048]}, {"id": "unconnected", "fg": 1054}]}, {"id": "f_ash", "fg": 1057}, {"id": "f_gunsafe_ml", "fg": 1060}, {"id": "f_gunsafe_mj", "fg": 1058}, {"id": "f_gun_safe_el", "fg": 1059}, {"id": "f_mutpoppy", "fg": 1061}, {"id": "f_alien_anemone", "fg": 1062}, {"id": "f_alien_table", "fg": 1063}, {"id": "f_stool", "fg": 1064}, {"id": "f_firering", "fg": 1065}, {"id": "f_bed", "multitile": true, "fg": 1079, "additional_tiles": [{"id": "center", "fg": 1075}, {"id": "corner", "fg": [1077, 1081, 1069, 1066]}, {"id": "t_connection", "fg": [1076, 1067, 1072, 1078]}, {"id": "edge", "fg": [1074, 1071]}, {"id": "end_piece", "fg": [1080, 1070, 1068, 1073]}, {"id": "unconnected", "fg": 1079}]}, {"id": "f_planter_harvest", "multitile": true, "fg": 1096, "additional_tiles": [{"id": "center", "fg": 1083}, {"id": "corner", "fg": [1090, 1094, 1093, 1091]}, {"id": "t_connection", "fg": [1088, 1087, 1086, 1089]}, {"id": "edge", "fg": [1085, 1097]}, {"id": "end_piece", "fg": [1092, 1095, 1082, 1084]}, {"id": "unconnected", "fg": 1096}]}, {"id": "f_desk", "multitile": true, "fg": 1105, "additional_tiles": [{"id": "center", "fg": 1114}, {"id": "corner", "fg": [1113, 1101, 1098, 1099]}, {"id": "t_connection", "fg": [1110, 1102, 1107, 1104]}, {"id": "edge", "fg": [1103, 1108]}, {"id": "end_piece", "fg": [1106, 1111, 1112, 1100]}, {"id": "unconnected", "fg": 1105}]}, {"id": "f_filing_cabinet", "fg": 1115}, {"id": "f_dandelion", "fg": 1116}, {"id": "f_trashcan", "fg": 1117}, {"id": "f_chair", "fg": 1118}, {"id": "f_dahlia", "fg": [{"weight": 1, "sprite": 1120}, {"weight": 2, "sprite": 1119}]}, {"id": "f_grave_stone_old", "fg": [{"weight": 1, "sprite": 1121}, {"weight": 1, "sprite": 1122}]}, {"id": "f_planter_mature", "multitile": true, "fg": 1123, "additional_tiles": [{"id": "center", "fg": 1129}, {"id": "corner", "fg": [1124, 1131, 1130, 1126]}, {"id": "t_connection", "fg": [1138, 1134, 1136, 1125]}, {"id": "edge", "fg": [1135, 1128]}, {"id": "end_piece", "fg": [1137, 1127, 1132, 1133]}, {"id": "unconnected", "fg": 1123}]}, {"id": "f_indoor_plant", "fg": 1140}, {"id": "f_indoor_plant_y", "fg": 1141}, {"id": ["f_indoor_plant_y_season_autumn", "f_indoor_plant_y_season_winter"], "fg": 1139}, {"id": "f_grave_stone", "fg": [{"weight": 1, "sprite": 1142}, {"weight": 1, "sprite": 1143}]}, {"id": "f_armchair", "fg": 1144}, {"id": "f_datura", "fg": 1145}, {"id": "f_air_conditioner", "fg": 1146}, {"id": "f_chamomile", "fg": 1147}, {"id": "f_flower_tulip", "fg": [{"weight": 1, "sprite": 1148}, {"weight": 2, "sprite": 1149}]}, {"id": "f_cardboard_box", "fg": 1150}, {"id": "f_boulder_large", "fg": 1151}, {"id": "f_toilet", "fg": 1152}, {"id": "f_bench", "multitile": true, "fg": 1166, "additional_tiles": [{"id": "center", "fg": 1168}, {"id": "corner", "fg": [1158, 1163, 1161, 1167]}, {"id": "t_connection", "fg": [1165, 1157, 1156, 1162]}, {"id": "edge", "fg": [1164, 1160]}, {"id": "end_piece", "fg": [1153, 1154, 1155, 1159]}, {"id": "unconnected", "fg": 1166}]}, {"id": "f_entertainment_center", "fg": 1169}, {"id": "f_planter_seedling", "multitile": true, "fg": 1178, "additional_tiles": [{"id": "center", "fg": 1183}, {"id": "corner", "fg": [1181, 1173, 1176, 1177]}, {"id": "t_connection", "fg": [1179, 1185, 1171, 1184]}, {"id": "edge", "fg": [1170, 1174]}, {"id": "end_piece", "fg": [1182, 1175, 1172, 1180]}, {"id": "unconnected", "fg": 1178}]}, {"id": "f_rack", "fg": 1186}, {"id": "f_sofa", "multitile": true, "fg": 1195, "additional_tiles": [{"id": "center", "fg": 1188}, {"id": "corner", "fg": [1191, 1194, 1190, 1192]}, {"id": "t_connection", "fg": [1198, 1201, 1193, 1200]}, {"id": "edge", "fg": [1199, 1196]}, {"id": "end_piece", "fg": [1187, 1197, 1202, 1189]}, {"id": "unconnected", "fg": 1195}]}, {"id": "f_bluebell", "fg": [{"weight": 1, "sprite": 1203}, {"weight": 2, "sprite": 1204}]}, {"id": "f_planter", "multitile": true, "fg": 1214, "additional_tiles": [{"id": "center", "fg": 1215}, {"id": "corner", "fg": [1206, 1211, 1208, 1219]}, {"id": "t_connection", "fg": [1210, 1220, 1212, 1213]}, {"id": "edge", "fg": [1216, 1205]}, {"id": "end_piece", "fg": [1207, 1218, 1217, 1209]}, {"id": "unconnected", "fg": 1214}]}, {"id": "f_flower_spurge", "fg": 1221}, {"id": "f_mailbox", "fg": 1222}, {"id": "f_hay", "fg": 1223}, {"id": "tr_nailboard", "fg": 1224, "bg": 1304}, {"id": "mon_piglet", "fg": 1225}, {"id": "mon_zombie_pig", "fg": 1226}, {"id": "mon_ant", "fg": 1229}, {"id": "mon_ant_acid", "fg": 1227}, {"id": "corpse_mon_ant", "fg": 1234}, {"id": "corpse_mon_ant_acid", "fg": 1233}, {"id": "mon_ant_larva", "fg": 1228}, {"id": "mon_ant_acid_larva", "fg": 1230}, {"id": "corpse_mon_ant_larva", "fg": 1232}, {"id": "corpse_mon_ant_acid_larva", "fg": 1231}, {"id": "mon_zombeaver", "fg": 1235}, {"id": "mon_plague_nymph", "fg": 1236}, {"id": "mon_duck", "fg": 1237}, {"id": "mon_dermatik_larva", "fg": 1238}, {"id": "mon_locust_nymph", "fg": 1239}, {"id": "mon_mosquito", "fg": 1240}, {"id": "mon_zolf", "fg": 1241}, {"id": "mon_dog", "fg": 1246}, {"id": "mon_dog_skeleton", "fg": 1242}, {"id": "mon_zombie_dog", "fg": [{"weight": 1, "sprite": 1245}, {"weight": 1, "sprite": 1247}]}, {"id": "mon_dog_beagle", "fg": 1243}, {"id": "mon_dog_gshepherd", "fg": 1249}, {"id": "mon_dog_boxer", "fg": 1244}, {"id": "mon_dog_dachshund", "fg": 1248}, {"id": "mon_otter", "fg": 1250}, {"id": "mon_locust", "fg": 1251}, {"id": "mon_squirrel_red", "fg": 1252}, {"id": "mon_skittering_plague", "fg": 1253}, {"id": "mon_beaver", "fg": 1254}, {"id": "mon_pregnant_giant_cockroach", "fg": 1255}, {"id": "mon_squirrel", "fg": 1256}, {"id": "mon_wolf", "fg": 1257}, {"id": "mon_centipede_giant", "fg": 1258}, {"id": "mon_plague_vector", "fg": 1259}, {"id": "mon_deer_mouse", "fg": 1260}, {"id": "mon_pig", "fg": [{"weight": 8, "sprite": 1262}, {"weight": 3, "sprite": 1263}, {"weight": 1, "sprite": 1261}]}, {"id": "mon_rabbit", "fg": 1264}, {"id": "mon_bear_cub", "fg": 1265}, {"id": "mon_black_rat", "fg": 1266}, {"id": "mon_dragonfly", "fg": 1267}, {"id": "mon_zombie_child", "fg": [{"weight": 1, "sprite": 1269}, {"weight": 1, "sprite": 1268}]}, {"id": "mon_giant_cockroach", "fg": 1270}, {"id": "mon_wasp_small", "fg": 1271}, {"id": "mon_giant_cockroach_nymph", "fg": 1272}, {"id": "fd_fire", "fg": 1273}, {"id": "spoon", "fg": 1274}, {"id": "glass_shard", "fg": 1275}, {"id": "jug_plastic", "fg": 1276}, {"id": "knife_butter", "fg": 1277}, {"id": "9mm_casing", "fg": 1278}, {"id": "cup_plastic", "fg": 1279}, {"id": "can_drink_unsealed", "fg": 1280}, {"id": "pan", "fg": 1281}, {"id": "thermos", "fg": 1282}, {"id": "knife_vegetable_cleaver", "fg": 1283}, {"id": "welder", "fg": 1284}, {"id": "ash", "fg": 1285}, {"id": "blanket", "fg": 1286}, {"id": "down_blanket", "fg": 1287}, {"id": "plastic_sheet", "fg": 1288}, {"id": "pot_canning", "fg": 1289}, {"id": "jar_glass_sealed", "fg": 1290}, {"id": "bat_metal", "fg": 1291}, {"id": "", "fg": []}, {"id": "mp5mag", "fg": 1292}, {"id": "contacts", "fg": 1293}, {"id": "hammer", "fg": 1294}, {"id": "mop", "fg": 1295}, {"id": "straw_pile", "fg": 1296}, {"id": "sewing_kit", "fg": 1297}, {"id": ["rifle_9mm", "rifle_3006", "rifle_45", "rifle_22", "rifle_40", "rifle_44", "rifle_38", "rifle_223"], "fg": 1298}, {"id": "m79", "fg": 1299}, {"id": "glass_plate", "fg": 1300}, {"id": "pool_ball", "fg": 1301}, {"id": "wheat", "fg": 1302}, {"id": "slingshot", "fg": 1303}, {"id": "board_trap", "fg": 1304}, {"id": "pointy_stick", "fg": 1313}, {"id": "spear_wood", "fg": 1312}, {"id": "spear_spike", "fg": 1306}, {"id": "spear_knife", "fg": 1307}, {"id": "spear_knife_superior", "fg": 1309}, {"id": "spear_pipe", "fg": 1310}, {"id": "spear_rebar", "fg": 1308}, {"id": "spear_steel", "fg": 1305}, {"id": "spear_copper", "fg": 1311}, {"id": "scrap", "fg": 1314}, {"id": "nail", "fg": 1315}, {"id": "2x4", "fg": 1316}, {"id": "scissors", "fg": 1317}, {"id": "bag_canvas", "fg": 1318}, {"id": "file", "fg": 1319}, {"id": "wood_sheet", "fg": 1320}, {"id": "rag", "fg": 1321}, {"id": ["seed_hops", "seed_blackberries", "seed_blueberries", "coffee_pod", "seed_wheat", "roasted_coffee_bean", "seed_chamomile", "coffee_bean", "seed_celery", "fried_seeds", "seed_buckwheat", "seed_oats", "seed_mushroom_morel", "datura_seed", "seed_mushroom", "seed_rhubarb", "seed_wild_herbs", "seed_raw_dandelion", "seed_veggy_wild", "seed_chili_pepper", "seed_dogbane", "seed_mugwort", "seed_bee_balm", "seed_flower", "seed_sunflower", "seed_thyme", "seed_canola", "seed_pumpkin", "seed_beans", "seed_lentils", "soybean_seed", "marloss_seed", "fungal_seeds", "seed_weed", "seed_potato_raw", "seed_cucumber", "seed_corn", "seed_carrot", "garlic_clove", "seed_cactus", "seed_wildcarrot", "seed_cattail", "seed_chicory", "seed_salsify_raw", "seed_dahlia", "seed_garlic", "seed_broccoli", "seed_onion", "seed_zucchini", "seed_cotton_boll", "seed_tomato", "seed_rose", "seed_lettuce", "seed_cabbage", "seed_sugar_beet", "seed_tobacco", "seed_barley", "seed_grapes", "seed_strawberries", "seed_raspberries", "seed_cranberries", "seed_huckleberries", "seed_mulberries", "seed_elderberries"], "fg": 1322}, {"id": "pipe", "fg": 1323}, {"id": "crucible_clay", "fg": 1324}, {"id": "bag_zipper", "fg": 1325}, {"id": "can_drink", "fg": 1326}, {"id": "pine_bough", "fg": 1327}, {"id": "bottle_plastic", "fg": 1328}, {"id": "primitive_hammer", "fg": 1329}, {"id": "jar_glass", "fg": 1330}, {"id": "cattlefodder", "fg": 1331}, {"id": "chips", "fg": 1332}, {"id": "bottle_glass", "fg": 1333}, {"id": "box_medium", "fg": 1334}, {"id": "many_years_old_newspaper", "fg": 1338}, {"id": "months_old_newspaper", "fg": 1336}, {"id": "newest_newspaper", "fg": 1335}, {"id": "one_year_old_newspaper", "fg": 1339}, {"id": "weeks_old_newspaper", "fg": 1337}, {"id": "flashlight", "fg": 1340}, {"id": "heavy_flashlight", "fg": 1341}, {"id": "knife_butcher", "fg": 1342}, {"id": "usb_drive", "fg": 1343}, {"id": "bolt_cf", "fg": 1360}, {"id": "bolt_explosive", "fg": 1361}, {"id": "bolt_metal", "fg": 1356}, {"id": ["bolt_steel", "bolt_steel_bodkin", "bolt_steel_target"], "fg": 1351}, {"id": ["bolt_wood", "bolt_crude", "bolt_simple_wood", "bolt_simple_small_game", "bolt_makeshift", "bolt_wood_bodkin", "bolt_wood_small_game"], "fg": 1344}, {"id": "arrow_cf", "fg": 1357}, {"id": "arrow_exploding", "fg": 1359}, {"id": "arrow_field_point_fletched", "fg": 1345}, {"id": "arrow_fire_hardened_fletched", "fg": 1348}, {"id": "arrow_flammable", "fg": 1355}, {"id": "arrow_flamming", "fg": 1354}, {"id": "arrow_heavy_fire_hardened_fletched", "fg": 1347}, {"id": "arrow_metal", "fg": 1349}, {"id": "arrow_metal_sharpened_fletched", "fg": 1346}, {"id": "arrow_plastic", "fg": 1353}, {"id": "arrow_small_game_fletched", "fg": 1358}, {"id": "arrow_wood", "fg": 1352}, {"id": "arrow_wood_heavy", "fg": 1350}, {"id": "lighter", "fg": [{"weight": 1, "sprite": 1363}, {"weight": 1, "sprite": 1362}, {"weight": 1, "sprite": 1364}]}, {"id": "box_small", "fg": 1365}, {"id": "rock", "fg": 1366}, {"id": "splinter", "fg": 1367}, {"id": "pinecone", "fg": 1368}, {"id": "shot_hull", "fg": 1369}, {"id": "hickory_nut", "fg": 1370}, {"id": "ar15", "fg": 1371}, {"id": "hacksaw", "fg": 1372}, {"id": "chainsaw_off", "fg": 1373}, {"id": "makeshift_crowbar", "fg": 1374}, {"id": "id_science", "fg": 1375}, {"id": "crossbow", "fg": 1376}, {"id": "brick", "fg": 1377}, {"id": "arming_sword", "fg": 1378}, {"id": "backpack", "fg": 1379}, {"id": "rebar", "fg": 1380}, {"id": "pot", "fg": 1381}, {"id": "wrapper", "fg": 1382}, {"id": "wood_panel", "fg": 1383}, {"id": "tailors_kit", "fg": 1384}, {"id": "bwirebat", "fg": 1385}, {"id": "sheet_metal_small", "fg": 1386}, {"id": "bag_plastic", "fg": 1387}, {"id": "bat", "fg": 1388}, {"id": "crowbar", "fg": 1389}, {"id": "longbow", "fg": 1390}, {"id": "filter_air", "fg": 1391}, {"id": "cable", "fg": 1392}, {"id": "rolling_pin", "fg": 1393}, {"id": "matches", "fg": 1394}, {"id": "sheet_metal", "fg": 1395}, {"id": "paperback_novel", "fg": 1421}, {"id": "novel_adventure", "fg": 1421}, {"id": "novel_buddy", "fg": 1401}, {"id": "novel_coa", "fg": 1414}, {"id": "novel_coa2", "fg": 1411}, {"id": "novel_crime", "fg": 1420}, {"id": "novel_crime2", "fg": 1415}, {"id": "novel_drama", "fg": 1409}, {"id": "novel_erotic", "fg": 1397}, {"id": "novel_experimental", "fg": 1403}, {"id": "novel_fantasy", "fg": 1398}, {"id": "novel_horror", "fg": 1408}, {"id": "novel_mystery", "fg": 1419}, {"id": "novel_pulp", "fg": 1404}, {"id": "novel_road", "fg": 1413}, {"id": "novel_romance", "fg": 1412}, {"id": "novel_samurai", "fg": 1399}, {"id": "novel_satire", "fg": 1417}, {"id": "novel_scifi", "fg": 1407}, {"id": "novel_sports", "fg": 1406}, {"id": "novel_spy", "fg": 1396}, {"id": "novel_swash", "fg": 1400}, {"id": "novel_thriller", "fg": 1416}, {"id": "novel_tragedy", "fg": 1402}, {"id": "novel_war", "fg": 1410}, {"id": "novel_war2", "fg": 1405}, {"id": "novel_western", "fg": 1418}, {"id": "jar_3l_glass_sealed", "fg": 1422}, {"id": "40mm_casing", "fg": 1423}, {"id": "id_military", "fg": 1424}, {"id": "can_food", "fg": 1425}, {"id": "1st_aid", "fg": 1426}, {"id": "television", "fg": 1427}, {"id": "nailbat", "fg": 1428}, {"id": "pillow", "fg": 1430}, {"id": "down_pillow", "fg": 1429}, {"id": "apple", "fg": 1431}, {"id": "banana", "fg": 1436}, {"id": "broccoli", "fg": 1443}, {"id": "corn", "fg": 1439}, {"id": "cucumber", "fg": 1437}, {"id": "egg_bird", "fg": 1444}, {"id": "grapes", "fg": 1432}, {"id": "lemon", "fg": 1441}, {"id": "onion", "fg": 1440}, {"id": "orange", "fg": 1438}, {"id": "pear", "fg": 1434}, {"id": "potato", "fg": 1435}, {"id": "pumpkin", "fg": 1433}, {"id": "tomato", "fg": 1442}, {"id": "stick_long", "fg": 1445}, {"id": "broom", "fg": 1446}, {"id": ["glock_17", "glock_19", "glock_18c", "glock_22", "glock_31"], "fg": 1447}, {"id": "ak74", "fg": 1448}, {"id": "aspirin", "fg": 1461}, {"id": "bandages", "fg": 1454}, {"id": "syringe", "fg": 1462}, {"id": "antibiotics", "fg": 1463}, {"id": "weak_antibiotic", "fg": 1449}, {"id": "strong_antibiotic", "fg": 1455}, {"id": "vitamins", "fg": 1460}, {"id": "gummy_vitamins", "fg": 1459}, {"id": "calcium_tablet", "fg": 1457}, {"id": "oxycodone", "fg": 1452}, {"id": "tramadol", "fg": 1451}, {"id": "codeine", "fg": 1450}, {"id": "prussian_blue", "fg": 1464}, {"id": "iodine", "fg": 1456}, {"id": "antiparasitic", "fg": 1458}, {"id": "antifungal", "fg": 1453}, {"id": "steel_lump", "fg": 1465}, {"id": "bowl_plastic", "fg": 1466}, {"id": "box_cigarette", "fg": 1467}, {"id": "teapot", "fg": 1468}, {"id": "steel_chunk", "fg": 1469}, {"id": "marble", "fg": 1470}, {"id": "sponge", "fg": 1471}, {"id": "juniper", "fg": 1472}, {"id": "pipe_shotgun", "fg": 1473}, {"id": "id_industrial", "fg": 1474}, {"id": "fp_loyalty_card", "fg": 1475}, {"id": "bottle_plastic_small", "fg": 1476}, {"id": "wrench", "fg": 1477}, {"id": "withered", "fg": 1478}, {"id": "jar_3l_glass", "fg": 1479}, {"id": "shovel", "fg": 1480}, {"id": "50_casing", "fg": 1481}, {"id": "remington_870", "fg": 1482}, {"id": "stick", "fg": 1483}, {"id": "saw", "fg": 1484}, {"id": "log", "fg": 1485}, {"id": "fork", "fg": 1486}, {"id": "rifle_flintlock", "fg": 1487}, {"id": "cig_butt", "fg": 1488}, {"id": "sharp_rock", "fg": 1489}, {"id": "plastic_shopping_bag", "fg": 1490}, {"id": "wire", "fg": 1491}, {"id": "acorns", "fg": 1492}, {"id": "coffeemaker", "fg": 1493}, {"id": "welder_crude", "fg": 1494}, {"id": "water", "fg": 1495}, {"id": "water_clean", "fg": 1497}, {"id": "blood", "fg": 1496}, {"id": "box_large", "fg": 1498}, {"id": "meat", "fg": 1499}, {"id": "cash_card", "fg": 1500}, {"id": "screwdriver", "fg": 1501}, {"id": "fire_ax", "fg": 1502}, {"id": "ax", "fg": 1503}, {"id": "hatchet", "fg": 1504}, {"id": "balclava", "fg": 1512}, {"id": "beret", "fg": 1514}, {"id": "boots", "fg": 1523}, {"id": "boxer_briefs", "fg": 1549}, {"id": "boxer_shorts", "fg": 1517}, {"id": "boy_shorts", "fg": 1539}, {"id": "bra", "fg": 1540}, {"id": "briefs", "fg": 1548}, {"id": "cowboy_hat", "fg": 1533}, {"id": "dress_shoes", "fg": 1525}, {"id": "hat_ball", "fg": 1536}, {"id": "hat_cotton", "fg": 1508}, {"id": "hat_fur", "fg": 1546}, {"id": "hat_knit", "fg": 1518}, {"id": "hat_noise_cancelling", "fg": 1505}, {"id": "helmet_army", "fg": 1532}, {"id": "helmet_barbute", "fg": 1526}, {"id": "helmet_chitin", "fg": 1520}, {"id": "helmet_kabuto", "fg": 1543}, {"id": "hoodie", "fg": 1528}, {"id": "jeans", "fg": 1519}, {"id": "longshirt", "fg": 1550}, {"id": "maid_dress", "fg": 1535}, {"id": "maid_hat", "fg": 1524}, {"id": "mask_dust", "fg": 1537}, {"id": "panties", "fg": 1511}, {"id": "pants", "fg": 1516}, {"id": "pants_cargo", "fg": 1529}, {"id": "polo_shirt", "fg": 1522}, {"id": "ragpouch", "fg": 1521}, {"id": "sneakers", "fg": 1545}, {"id": "socks", "fg": 1541}, {"id": "stockings", "fg": 1506}, {"id": "sweater", "fg": 1510}, {"id": "sweatshirt", "fg": 1544}, {"id": "tank_top", "fg": 1527}, {"id": "tshirt", "fg": 1515}, {"id": "turban", "fg": 1547}, {"id": "undershirt", "fg": 1509}, {"id": "coat_lab", "fg": 1534}, {"id": "coat_rain", "fg": 1531}, {"id": "sports_bra", "fg": 1530}, {"id": "skirt", "fg": 1507}, {"id": "jacket_light", "fg": 1542}, {"id": "jacket_army", "fg": 1538}, {"id": "hat_hard", "fg": 1513}, {"id": "knife_meat_cleaver", "fg": 1551}, {"id": "pot_copper", "fg": 1552}, {"id": "corpse_generic_human", "fg": 1553}, {"id": "hickory_root", "fg": 1554}, {"id": "nailboard", "fg": 1555}, {"id": "223_casing", "fg": 1556}, {"id": ["overlay_effects_bleed"], "fg": 1564}, {"id": ["overlay_effects_deaf"], "fg": 1559}, {"id": ["overlay_effects_downed"], "fg": 1563}, {"id": ["overlay_effects_grabbed"], "fg": 1561}, {"id": ["overlay_effects_winded"], "fg": 1558}, {"id": ["overlay_effects_hot"], "fg": 1565}, {"id": ["overlay_effects_cold"], "fg": 1560}, {"id": ["overlay_male_crouch", "overlay_female_crouch"], "fg": 1566}, {"id": ["overlay_male_run", "overlay_female_run"], "fg": 1562}, {"id": ["overlay_hostile_sees_player"], "fg": 1567}, {"id": ["zombie_revival_indicator"], "fg": 1557}, {"id": "animation_hit", "fg": 1568}, {"id": "cursor", "fg": 1571}, {"id": "highlight", "fg": 1569}, {"id": "highlight_item", "fg": 1574}, {"id": "line_target", "fg": 1573}, {"id": "line_trail", "fg": 1572}, {"id": "animation_line", "fg": 1570}], "//": "range 1 to 1584"}, {"file": "tall.png", "tiles": [{"id": "t_machinery_heavy", "fg": 1585, "bg": 301}, {"id": "t_machinery_old", "fg": 1586, "bg": 818}, {"id": "t_stairs_up", "fg": 1588}, {"id": "t_wood_stairs_up", "fg": 1587}, {"id": "t_column", "fg": 1589}, {"id": "t_gutter_downspout", "fg": 1590}, {"id": "t_ladder_up", "fg": 1595}, {"id": "t_utility_light", "fg": 1596}, {"id": "t_atm", "fg": 1597}, {"id": "f_home_furnace", "fg": 1598}, {"id": "f_standing_tank", "fg": 1599}, {"id": "f_glass_cabinet", "fg": 1600}, {"id": "f_glass_fridge", "fg": 1601}, {"id": "f_dumpster", "multitile": true, "fg": 1603, "additional_tiles": [{"id": "edge", "fg": [1604, 1602]}, {"id": "end_piece", "fg": [1605, 1606, 1607, 1608]}, {"id": "unconnected", "fg": 1603}]}, {"id": "f_dresser", "fg": 1609}, {"id": "f_workbench", "fg": 1610}, {"id": "f_woodstove", "fg": 1611}, {"id": "f_dryer", "fg": 1612}, {"id": "f_alien_tendril", "fg": 1613}, {"id": "f_alien_zapper", "fg": 1615}, {"id": ["f_alien_pod", "f_alien_pod_organ"], "fg": 1614}, {"id": "f_alien_pod_resin", "fg": 1616}, {"id": "f_brazier", "fg": 1617}, {"id": "f_washer", "fg": 1618}, {"id": "f_crate_c", "fg": 1620}, {"id": "f_crate_o", "fg": 1619}, {"id": "f_oven", "fg": 1621}, {"id": "f_rack_coat", "fg": 1622}, {"id": "f_water_heater", "fg": 1623}, {"id": "f_birdbath", "fg": 1624}, {"id": "f_shower", "fg": 1625}, {"id": "f_floor_lamp", "fg": 1626}, {"id": "f_cattails_season_spring", "fg": 1627, "rotates": false}, {"id": "f_cattails_season_summer", "fg": 1630, "rotates": false}, {"id": "f_cattails_season_autumn", "fg": 1629, "rotates": false}, {"id": "f_cattails_season_winter", "fg": 1628, "rotates": false}, {"id": "f_bigmirror", "fg": 1631}, {"id": "f_bookcase", "fg": 1632}, {"id": "f_locker", "fg": 1633}, {"id": "f_arcade_machine", "fg": 1634}, {"id": "f_statue", "fg": 1635}, {"id": "player_male", "fg": 2132}, {"id": "player_female", "fg": 2131}, {"id": "npc_female", "fg": 2131}, {"id": "npc_male", "fg": 2132}, {"id": "mon_dragonfly_giant", "fg": 1636}, {"id": "mon_dermatik", "fg": 1637}, {"id": "mon_zombie_soldier", "fg": 1638}, {"id": "mon_skeleton", "fg": 1639}, {"id": "mon_cougar", "fg": 1640}, {"id": "mon_zombie_swimmer", "fg": 1641}, {"id": "mon_mosquito_giant", "fg": 1642}, {"id": "mon_zombie_brainless", "fg": 1643}, {"id": "mon_fungaloid", "fg": 1644}, {"id": "mon_bee", "fg": 1645}, {"id": "mon_fly", "fg": 1646}, {"id": "mon_zougar", "fg": 1647}, {"id": "overlay_male_mutation_hair_white_medium", "fg": 1649}, {"id": "overlay_female_mutation_hair_white_medium", "fg": 1648}, {"id": "overlay_male_mutation_hair_black_short", "fg": 1651}, {"id": "overlay_female_mutation_hair_black_short", "fg": 1650}, {"id": "overlay_male_mutation_hair_brown_mohawk", "fg": 1653}, {"id": "overlay_female_mutation_hair_brown_mohawk", "fg": 1652}, {"id": "overlay_male_mutation_hair_brown_medium", "fg": 1654}, {"id": "overlay_female_mutation_hair_brown_medium", "fg": 1655}, {"id": "overlay_male_mutation_hair_red_fro", "fg": 1657}, {"id": "overlay_female_mutation_hair_red_fro", "fg": 1656}, {"id": "overlay_male_mutation_hair_black_long", "fg": 1659}, {"id": "overlay_female_mutation_hair_black_long", "fg": 1658}, {"id": "overlay_male_mutation_hair_gray_short", "fg": 1661}, {"id": "overlay_female_mutation_hair_gray_short", "fg": 1660}, {"id": "overlay_male_mutation_hair_black_mohawk", "fg": 1663}, {"id": "overlay_female_mutation_hair_black_mohawk", "fg": 1662}, {"id": "overlay_male_mutation_hair_black_fro", "fg": 1665}, {"id": "overlay_female_mutation_hair_black_fro", "fg": 1664}, {"id": "overlay_male_mutation_hair_blond_crewcut", "fg": 1666}, {"id": "overlay_female_mutation_hair_blond_crewcut", "fg": 1667}, {"id": "overlay_male_mutation_hair_brown_short", "fg": 1668}, {"id": "overlay_female_mutation_hair_brown_short", "fg": 1669}, {"id": "overlay_male_mutation_hair_white_short", "fg": 1670}, {"id": "overlay_female_mutation_hair_white_short", "fg": 1671}, {"id": "overlay_male_mutation_hair_blond_medium", "fg": 1673}, {"id": "overlay_female_mutation_hair_blond_medium", "fg": 1672}, {"id": "overlay_male_mutation_hair_blond_fro", "fg": 1674}, {"id": "overlay_female_mutation_hair_blond_fro", "fg": 1675}, {"id": "overlay_male_mutation_hair_brown_fro", "fg": 1676}, {"id": "overlay_female_mutation_hair_brown_fro", "fg": 1677}, {"id": "overlay_male_mutation_hair_brown_long", "fg": 1679}, {"id": "overlay_female_mutation_hair_brown_long", "fg": 1678}, {"id": "overlay_male_mutation_hair_black_medium", "fg": 1680}, {"id": "overlay_female_mutation_hair_black_medium", "fg": 1681}, {"id": "overlay_male_mutation_hair_blond_mohawk", "fg": 1683}, {"id": "overlay_female_mutation_hair_blond_mohawk", "fg": 1682}, {"id": "overlay_male_mutation_hair_black_crewcut", "fg": 1684}, {"id": "overlay_female_mutation_hair_black_crewcut", "fg": 1685}, {"id": "overlay_male_mutation_hair_gray_fro", "fg": 1686}, {"id": "overlay_female_mutation_hair_gray_fro", "fg": 1687}, {"id": "overlay_male_mutation_hair_red_crewcut", "fg": 1689}, {"id": "overlay_female_mutation_hair_red_crewcut", "fg": 1688}, {"id": "overlay_male_mutation_hair_red_mohawk", "fg": 1690}, {"id": "overlay_female_mutation_hair_red_mohawk", "fg": 1691}, {"id": "overlay_male_mutation_hair_white_fro", "fg": 1692}, {"id": "overlay_female_mutation_hair_white_fro", "fg": 1693}, {"id": "overlay_male_mutation_hair_gray_mohawk", "fg": 1695}, {"id": "overlay_female_mutation_hair_gray_mohawk", "fg": 1694}, {"id": "overlay_male_mutation_hair_blond_short", "fg": 1697}, {"id": "overlay_female_mutation_hair_blond_short", "fg": 1696}, {"id": "overlay_male_mutation_hair_white_long", "fg": 1698}, {"id": "overlay_female_mutation_hair_white_long", "fg": 1699}, {"id": "overlay_male_mutation_hair_white_mohawk", "fg": 1700}, {"id": "overlay_female_mutation_hair_white_mohawk", "fg": 1701}, {"id": "overlay_male_mutation_hair_red_short", "fg": 1702}, {"id": "overlay_female_mutation_hair_red_short", "fg": 1703}, {"id": "overlay_male_mutation_hair_blond_long", "fg": 1705}, {"id": "overlay_female_mutation_hair_blond_long", "fg": 1704}, {"id": "overlay_male_mutation_hair_gray_crewcut", "fg": 1706}, {"id": "overlay_female_mutation_hair_gray_crewcut", "fg": 1707}, {"id": "overlay_male_mutation_hair_gray_medium", "fg": 1709}, {"id": "overlay_female_mutation_hair_gray_medium", "fg": 1708}, {"id": "overlay_male_mutation_hair_red_long", "fg": 1710}, {"id": "overlay_female_mutation_hair_red_long", "fg": 1711}, {"id": "overlay_male_mutation_hair_brown_crewcut", "fg": 1713}, {"id": "overlay_female_mutation_hair_brown_crewcut", "fg": 1712}, {"id": "overlay_male_mutation_hair_gray_long", "fg": 1714}, {"id": "overlay_female_mutation_hair_gray_long", "fg": 1715}, {"id": "overlay_male_mutation_hair_white_crewcut", "fg": 1716}, {"id": "overlay_female_mutation_hair_white_crewcut", "fg": 1717}, {"id": "overlay_male_mutation_hair_red_medium", "fg": 1718}, {"id": "overlay_female_mutation_hair_red_medium", "fg": 1719}, {"id": "overlay_female_worn_hat_ball", "fg": 1720}, {"id": "overlay_male_worn_hat_ball", "fg": 1721}, {"id": "overlay_male_worn_pants_cargo", "fg": 1722}, {"id": "overlay_female_worn_pants_cargo", "fg": 1723}, {"id": "overlay_male_worn_pants", "fg": 1725}, {"id": "overlay_female_worn_pants", "fg": 1724}, {"id": "overlay_female_worn_skirt", "fg": 1726}, {"id": "overlay_male_worn_skirt", "fg": 1727}, {"id": "overlay_male_worn_mittens", "fg": 1728}, {"id": "overlay_female_worn_mittens", "fg": 1729}, {"id": "overlay_female_worn_boots_combat", "fg": 1730}, {"id": "overlay_male_worn_boots_combat", "fg": 1731}, {"id": "overlay_female_worn_hat_noise_cancelling", "fg": 1732}, {"id": "overlay_male_worn_hat_noise_cancelling", "fg": 1733}, {"id": "overlay_female_worn_helmet_bike", "fg": 1735}, {"id": "overlay_male_worn_helmet_bike", "fg": 1734}, {"id": "overlay_male_worn_blanket", "fg": 1736}, {"id": "overlay_female_worn_blanket", "fg": 1738}, {"id": "overlay_male_worn_down_blanket", "fg": 1739}, {"id": "overlay_female_worn_down_blanket", "fg": 1737}, {"id": "overlay_male_worn_geta", "fg": 1740}, {"id": "overlay_female_worn_geta", "fg": 1741}, {"id": "overlay_female_worn_jeans_red", "fg": 1742}, {"id": "overlay_male_worn_jeans_red", "fg": 1743}, {"id": "overlay_male_worn_union_suit", "fg": 1744}, {"id": "overlay_female_worn_union_suit", "fg": 1745}, {"id": "overlay_female_worn_coat_rain", "fg": 1747}, {"id": "overlay_male_worn_coat_rain", "fg": 1746}, {"id": "overlay_female_worn_panties", "fg": 1749}, {"id": "overlay_male_worn_panties", "fg": 1748}, {"id": "overlay_female_worn_boots_rubber", "fg": 1751}, {"id": "overlay_male_worn_boots_rubber", "fg": 1750}, {"id": "overlay_female_worn_jacket_army", "fg": 1753}, {"id": "overlay_male_worn_jacket_army", "fg": 1752}, {"id": "overlay_male_worn_welding_mask_crude_raised", "fg": 1755}, {"id": "overlay_female_worn_welding_mask_crude_raised", "fg": 1754}, {"id": "overlay_male_worn_sundress", "fg": 1757}, {"id": "overlay_female_worn_sundress", "fg": 1756}, {"id": "overlay_female_worn_tank_top", "fg": 1758}, {"id": "overlay_male_worn_tank_top", "fg": 1759}, {"id": "overlay_female_worn_ragpouch", "fg": 1761}, {"id": "overlay_male_worn_ragpouch", "fg": 1760}, {"id": "overlay_female_worn_skirt_leather", "fg": 1762}, {"id": "overlay_male_worn_skirt_leather", "fg": 1763}, {"id": "overlay_male_worn_socks", "fg": 1765}, {"id": "overlay_female_worn_socks", "fg": 1764}, {"id": "overlay_male_worn_gloves_cut_resistant", "fg": 1767}, {"id": "overlay_female_worn_gloves_cut_resistant", "fg": 1766}, {"id": "overlay_female_worn_boots_steel", "fg": 1768}, {"id": "overlay_male_worn_boots_steel", "fg": 1769}, {"id": "overlay_male_worn_jumpsuit", "fg": 1770}, {"id": "overlay_female_worn_jumpsuit", "fg": 1771}, {"id": "overlay_female_worn_gloves_fingerless", "fg": 1773}, {"id": "overlay_male_worn_gloves_fingerless", "fg": 1772}, {"id": "overlay_female_worn_pants_fur", "fg": 1774}, {"id": "overlay_male_worn_pants_fur", "fg": 1775}, {"id": "overlay_male_worn_welding_mask_raised", "fg": 1777}, {"id": "overlay_female_worn_welding_mask_raised", "fg": 1776}, {"id": "overlay_female_worn_army_top", "fg": 1779}, {"id": "overlay_male_worn_army_top", "fg": 1778}, {"id": "overlay_male_worn_glasses_bal", "fg": 1780}, {"id": "overlay_female_worn_glasses_bal", "fg": 1781}, {"id": "overlay_female_worn_welding_mask_crude", "fg": 1782}, {"id": "overlay_male_worn_welding_mask_crude", "fg": 1783}, {"id": "overlay_female_worn_pants_faux_fur", "fg": 1784}, {"id": "overlay_male_worn_pants_faux_fur", "fg": 1785}, {"id": "overlay_male_worn_longshirt", "fg": 1786}, {"id": "overlay_female_worn_longshirt", "fg": 1787}, {"id": "overlay_male_worn_tshirt", "fg": 1789}, {"id": "overlay_female_worn_tshirt", "fg": 1788}, {"id": "overlay_male_worn_survivor_suit", "fg": 1791}, {"id": "overlay_female_worn_survivor_suit", "fg": 1790}, {"id": "overlay_female_worn_stockings", "fg": 1793}, {"id": "overlay_male_worn_stockings", "fg": 1792}, {"id": "overlay_female_worn_gloves_light", "fg": 1795}, {"id": "overlay_male_worn_gloves_light", "fg": 1794}, {"id": "overlay_male_worn_glasses_monocle", "fg": 1797}, {"id": "overlay_female_worn_glasses_monocle", "fg": 1796}, {"id": "overlay_female_worn_dinosuit", "fg": 1799}, {"id": "overlay_male_worn_dinosuit", "fg": 1798}, {"id": "overlay_male_worn_hat_cotton", "fg": 1800}, {"id": "overlay_female_worn_hat_cotton", "fg": 1801}, {"id": "overlay_female_worn_robofac_jumpsuit", "fg": 1803}, {"id": "overlay_male_worn_robofac_jumpsuit", "fg": 1802}, {"id": "overlay_male_worn_shoes_bowling", "fg": 1805}, {"id": "overlay_female_worn_shoes_bowling", "fg": 1804}, {"id": "overlay_female_worn_armor_samurai", "fg": 1807}, {"id": "overlay_male_worn_armor_samurai", "fg": 1806}, {"id": "overlay_male_worn_turban", "fg": 1809}, {"id": "overlay_female_worn_turban", "fg": 1808}, {"id": "overlay_male_worn_hat_knit", "fg": 1811}, {"id": "overlay_female_worn_hat_knit", "fg": 1810}, {"id": "overlay_female_worn_rucksack", "fg": 1813}, {"id": "overlay_male_worn_rucksack", "fg": 1812}, {"id": "overlay_male_worn_cargo_pants", "fg": 1814}, {"id": "overlay_female_worn_sweater", "fg": 1816}, {"id": "overlay_male_worn_sweater", "fg": 1815}, {"id": "overlay_female_worn_helmet_army", "fg": 1817}, {"id": "overlay_male_worn_helmet_army", "fg": 1818}, {"id": "overlay_female_worn_maid_hat", "fg": 1820}, {"id": "overlay_female_worn_maid_dress", "fg": 1819}, {"id": "overlay_male_worn_maid_hat", "fg": 1822}, {"id": "overlay_male_worn_maid_dress", "fg": 1821}, {"id": "overlay_male_worn_glasses_reading", "fg": 1823}, {"id": "overlay_female_worn_glasses_reading", "fg": 1824}, {"id": "overlay_female_worn_sneakers", "fg": 1825}, {"id": "overlay_male_worn_sneakers", "fg": 1826}, {"id": "overlay_female_worn_jacket_flannel", "fg": 1827}, {"id": "overlay_male_worn_jacket_flannel", "fg": 1828}, {"id": "overlay_male_worn_boots_winter", "fg": 1830}, {"id": "overlay_female_worn_boots_winter", "fg": 1829}, {"id": "overlay_female_worn_ski_jacket", "fg": 1832}, {"id": "overlay_male_worn_ski_jacket", "fg": 1831}, {"id": "overlay_female_worn_shorts", "fg": 1833}, {"id": "overlay_male_worn_shorts", "fg": 1834}, {"id": "overlay_male_worn_long_glove_white", "fg": 1836}, {"id": "overlay_female_worn_long_glove_white", "fg": 1835}, {"id": "overlay_male_worn_hoodie", "fg": 1838}, {"id": "overlay_female_worn_hoodie", "fg": 1837}, {"id": "overlay_female_worn_bunker_pants", "fg": 1840}, {"id": "overlay_male_worn_bunker_pants", "fg": 1839}, {"id": "overlay_worn_jacket_jean", "fg": 1841}, {"id": "overlay_male_worn_boots_fur", "fg": 1843}, {"id": "overlay_female_worn_boots_fur", "fg": 1842}, {"id": "overlay_female_worn_gloves_medical", "fg": 1845}, {"id": "overlay_male_worn_gloves_medical", "fg": 1844}, {"id": "overlay_male_worn_arm_warmers", "fg": 1847}, {"id": "overlay_female_worn_arm_warmers", "fg": 1846}, {"id": "overlay_female_worn_helmet_barbute", "fg": 1848}, {"id": "overlay_male_worn_helmet_barbute", "fg": 1849}, {"id": "overlay_female_worn_boy_shorts", "fg": 1850}, {"id": "overlay_male_worn_boy_shorts", "fg": 1851}, {"id": "overlay_male_worn_gloves_work", "fg": 1853}, {"id": "overlay_female_worn_gloves_work", "fg": 1852}, {"id": "overlay_female_worn_flip_flops", "fg": 1854}, {"id": "overlay_male_worn_flip_flops", "fg": 1855}, {"id": "overlay_female_worn_hazmat_suit", "fg": 1857}, {"id": "overlay_male_worn_hazmat_suit", "fg": 1856}, {"id": "overlay_female_worn_firehelmet", "fg": 1859}, {"id": "overlay_male_worn_firehelmet", "fg": 1858}, {"id": "overlay_female_worn_pants_checkered", "fg": 1861}, {"id": "overlay_male_worn_pants_checkered", "fg": 1860}, {"id": "overlay_male_worn_bunker_coat", "fg": 1863}, {"id": "overlay_female_worn_bunker_coat", "fg": 1862}, {"id": "overlay_female_worn_jeans", "fg": 1864}, {"id": "overlay_male_worn_jeans", "fg": 1865}, {"id": "overlay_female_worn_bandana", "fg": 1866}, {"id": "overlay_male_worn_bandana", "fg": 1867}, {"id": "overlay_male_worn_boots_hiking", "fg": 1868}, {"id": "overlay_female_worn_boots_hiking", "fg": 1869}, {"id": "overlay_female_worn_beekeeping_hood", "fg": 1871}, {"id": "overlay_male_worn_beekeeping_hood", "fg": 1870}, {"id": "overlay_female_worn_pants_army", "fg": 1873}, {"id": "overlay_male_worn_pants_army", "fg": 1872}, {"id": "overlay_female_worn_gloves_fur", "fg": 1875}, {"id": "overlay_male_worn_gloves_fur", "fg": 1874}, {"id": "overlay_female_worn_mask_dust", "fg": 1877}, {"id": "overlay_male_worn_mask_dust", "fg": 1876}, {"id": "overlay_male_worn_glasses_bifocal", "fg": 1879}, {"id": "overlay_female_worn_glasses_bifocal", "fg": 1878}, {"id": "overlay_male_worn_kevlar", "fg": 1880}, {"id": "overlay_female_worn_kevlar", "fg": 1881}, {"id": "overlay_female_worn_jacket_light", "fg": 1882}, {"id": "overlay_male_worn_jacket_light", "fg": 1883}, {"id": "overlay_female_worn_sunglasses", "fg": 1885}, {"id": "overlay_male_worn_sunglasses", "fg": 1884}, {"id": "overlay_female_worn_coat_winter", "fg": 1887}, {"id": "overlay_male_worn_coat_winter", "fg": 1886}, {"id": "overlay_male_worn_backpack_leather", "fg": 1888}, {"id": "overlay_female_worn_backpack_leather", "fg": 1889}, {"id": "overlay_male_worn_gloves_liner", "fg": 1890}, {"id": "overlay_female_worn_gloves_liner", "fg": 1891}, {"id": "overlay_male_worn_gloves_tactical", "fg": 1892}, {"id": "overlay_female_worn_gloves_tactical", "fg": 1893}, {"id": "overlay_female_worn_gloves_leather", "fg": 1894}, {"id": "overlay_male_worn_gloves_leather", "fg": 1895}, {"id": "overlay_male_worn_shorts_cargo", "fg": 1896}, {"id": "overlay_female_worn_shorts_cargo", "fg": 1897}, {"id": "overlay_female_worn_duster", "fg": 1899}, {"id": "overlay_male_worn_duster", "fg": 1898}, {"id": "overlay_male_worn_helmet_motor", "fg": 1900}, {"id": "overlay_female_worn_helmet_motor", "fg": 1901}, {"id": "overlay_male_worn_sweatshirt", "fg": 1903}, {"id": "overlay_female_worn_sweatshirt", "fg": 1902}, {"id": "overlay_male_worn_boxer_briefs", "fg": 1905}, {"id": "overlay_female_worn_boxer_briefs", "fg": 1904}, {"id": "overlay_female_worn_gloves_rubber", "fg": 1906}, {"id": "overlay_male_worn_gloves_rubber", "fg": 1907}, {"id": "overlay_female_worn_gloves_winter", "fg": 1909}, {"id": "overlay_male_worn_gloves_winter", "fg": 1908}, {"id": "overlay_female_worn_sports_bra", "fg": 1910}, {"id": "overlay_male_worn_sports_bra", "fg": 1911}, {"id": "overlay_female_worn_boots", "fg": 1912}, {"id": "overlay_male_worn_boots", "fg": 1913}, {"id": "overlay_male_worn_bra", "fg": 1915}, {"id": "overlay_female_worn_bra", "fg": 1914}, {"id": "overlay_male_worn_boots_bunker", "fg": 1917}, {"id": "overlay_female_worn_boots_bunker", "fg": 1916}, {"id": "overlay_worn_dress_shoes", "fg": 1918}, {"id": "overlay_female_worn_helmet_chitin", "fg": 1919}, {"id": "overlay_male_worn_helmet_chitin", "fg": 1920}, {"id": "overlay_female_worn_beret", "fg": 1921}, {"id": "overlay_male_worn_beret", "fg": 1922}, {"id": "overlay_female_worn_pants_leather", "fg": 1924}, {"id": "overlay_male_worn_pants_leather", "fg": 1923}, {"id": "overlay_male_worn_glove_jackson", "fg": 1925}, {"id": "overlay_female_worn_glove_jackson", "fg": 1926}, {"id": "overlay_male_worn_scarf", "fg": 1928}, {"id": "overlay_female_worn_scarf", "fg": 1927}, {"id": "overlay_male_worn_glasses_eye", "fg": 1930}, {"id": "overlay_female_worn_glasses_eye", "fg": 1929}, {"id": "overlay_female_worn_shorts_denim", "fg": 1932}, {"id": "overlay_male_worn_shorts_denim", "fg": 1931}, {"id": "overlay_male_worn_pants_ski", "fg": 1934}, {"id": "overlay_female_worn_pants_ski", "fg": 1933}, {"id": "overlay_female_worn_gloves_golf", "fg": 1936}, {"id": "overlay_male_worn_gloves_golf", "fg": 1935}, {"id": "overlay_female_worn_dress_shirt", "fg": 1938}, {"id": "overlay_male_worn_dress_shirt", "fg": 1937}, {"id": "overlay_female_worn_motorbike_boots", "fg": 1939}, {"id": "overlay_male_worn_motorbike_boots", "fg": 1940}, {"id": "overlay_female_worn_cowboy_hat", "fg": 1942}, {"id": "overlay_male_worn_cowboy_hat", "fg": 1941}, {"id": "overlay_male_worn_balclava", "fg": 1943}, {"id": "overlay_female_worn_balclava", "fg": 1944}, {"id": "overlay_female_worn_boxer_shorts", "fg": 1946}, {"id": "overlay_male_worn_boxer_shorts", "fg": 1945}, {"id": "overlay_female_worn_welding_mask", "fg": 1948}, {"id": "overlay_male_worn_welding_mask", "fg": 1947}, {"id": "overlay_female_worn_helmet_kabuto", "fg": 1950}, {"id": "overlay_male_worn_helmet_kabuto", "fg": 1949}, {"id": "overlay_female_worn_glasses_safety", "fg": 1951}, {"id": "overlay_male_worn_glasses_safety", "fg": 1952}, {"id": "overlay_male_worn_touring_suit", "fg": 1954}, {"id": "overlay_female_worn_touring_suit", "fg": 1953}, {"id": "overlay_female_worn_motorbike_pants", "fg": 1955}, {"id": "overlay_male_worn_motorbike_pants", "fg": 1956}, {"id": "overlay_male_worn_undershirt", "fg": 1957}, {"id": "overlay_female_worn_hat_hard", "fg": 1959}, {"id": "overlay_male_worn_hat_hard", "fg": 1958}, {"id": "overlay_male_worn_hat_fur", "fg": 1960}, {"id": "overlay_female_worn_hat_fur", "fg": 1961}, {"id": "overlay_female_worn_coat_lab", "fg": 1962}, {"id": "overlay_male_worn_coat_lab", "fg": 1963}, {"id": "overlay_male_worn_leg_warmers", "fg": 1964}, {"id": "overlay_female_worn_leg_warmers", "fg": 1965}, {"id": "overlay_male_worn_boots_western", "fg": 1966}, {"id": "overlay_female_worn_boots_western", "fg": 1967}, {"id": "overlay_female_worn_gloves_wool", "fg": 1968}, {"id": "overlay_male_worn_gloves_wool", "fg": 1969}, {"id": "overlay_female_mutation_FANGS", "fg": 1971}, {"id": "overlay_male_mutation_FANGS", "fg": 1970}, {"id": "overlay_female_mutation_MUZZLE_LONG", "fg": 1972}, {"id": "overlay_male_mutation_MUZZLE_LONG", "fg": 1973}, {"id": "overlay_female_mutation_LIZ_EYE", "fg": 1974}, {"id": "overlay_male_mutation_LIZ_EYE", "fg": 1975}, {"id": "overlay_female_mutation_TALONS", "fg": 1976}, {"id": "overlay_male_mutation_TALONS", "fg": 1977}, {"id": "overlay_female_mutation_BIRD_EYE", "fg": 1978}, {"id": "overlay_male_mutation_BIRD_EYE", "fg": 1979}, {"id": "overlay_female_mutation_BEAK", "fg": 1981}, {"id": "overlay_male_mutation_BEAK", "fg": 1980}, {"id": "overlay_female_mutation_BEAK_PECK", "fg": 1982}, {"id": "overlay_male_mutation_BEAK_PECK", "fg": 1983}, {"id": "overlay_female_mutation_WINGS_BIRD", "fg": 1985}, {"id": "overlay_male_mutation_WINGS_BIRD", "fg": 1984}, {"id": "overlay_female_mutation_BEAK_HUM", "fg": 1987}, {"id": "overlay_male_mutation_BEAK_HUM", "fg": 1986}, {"id": "overlay_female_mutation_TAIL_THICK", "fg": 1989}, {"id": "overlay_male_mutation_TAIL_THICK", "fg": 1988}, {"id": "overlay_female_mutation_FEATHERS", "fg": 1991}, {"id": "overlay_male_mutation_FEATHERS", "fg": 1990}, {"id": "overlay_female_mutation_SCALES", "fg": 1993}, {"id": "overlay_male_mutation_SCALES", "fg": 1992}, {"id": "overlay_wielded_steel_glass_shard", "fg": 1994}, {"id": "overlay_wielded_jug_plastic", "fg": 1995}, {"id": "overlay_male_wielded_thermos", "fg": 1996}, {"id": "overlay_female_wielded_thermos", "fg": 1997}, {"id": ["overlay_wielded_glock_17", "overlay_wielded_glock_19", "overlay_wielded_glock_18c", "overlay_wielded_glock_22", "overlay_wielded_glock_31"], "fg": 1998}, {"id": "overlay_wielded_jar_glass_sealed", "fg": 1999}, {"id": ["overlay_wielded_corpse_mon_ant", "overlay_wielded_corpse_mon_ant_soldier", "overlay_wielded_corpse_mon_ant_queen"], "fg": 2002}, {"id": ["overlay_wielded_corpse_mon_ant_acid", "overlay_wielded_corpse_mon_ant_acid_soldier", "overlay_wielded_corpse_mon_ant_acid_queen"], "fg": 2003}, {"id": ["overlay_male_wielded_corpse_mon_zombie", "overlay_male_wielded_corpse_mon_zombie_grappler", "overlay_male_wielded_corpse_mon_zombie_biter", "overlay_male_wielded_corpse_mon_zombie_tough", "overlay_male_wielded_corpse_mon_zombie_hunter", "overlay_male_wielded_corpse_mon_zombie_brute", "overlay_male_wielded_corpse_mon_zombie_predator", "overlay_male_wielded_corpse_mon_zombie_necro", "overlay_male_wielded_corpse_mon_zombie_thorny", "overlay_male_wielded_corpse_mon_zombie_smoker", "overlay_male_wielded_corpse_mon_zombie_shady", "overlay_male_wielded_corpse_mon_zombie_master", "overlay_male_wielded_corpse_mon_zombie_acidic", "overlay_male_wielded_corpse_mon_zombie_fat", "overlay_male_wielded_corpse_mon_zombie_corrosive", "overlay_male_wielded_corpse_mon_zombie_screecher", "overlay_male_wielded_corpse_mon_zombie_scientist", "overlay_male_wielded_corpse_mon_zombie_runner", "overlay_male_wielded_corpse_mon_zombie_spitter", "overlay_male_wielded_corpse_mon_zombie_labsecurity", "overlay_male_wielded_corpse_mon_zombie_child", "overlay_male_wielded_corpse_mon_zombie_hulk", "overlay_male_wielded_corpse_mon_zombie_brute_grappler", "overlay_male_wielded_corpse_mon_zombie_brute_ninja", "overlay_male_wielded_corpse_mon_zombie_kevlar_2", "overlay_male_wielded_corpse_mon_zombie_kevlar_1", "overlay_male_wielded_corpse_mon_zombie_hazmat", "overlay_male_wielded_corpse_mon_zombie_electric", "overlay_male_wielded_corpse_mon_zombie_technician", "overlay_male_wielded_corpse_mon_zombie_fungus", "overlay_male_wielded_corpse_mon_zombie_cop", "overlay_male_wielded_corpse_mon_zombie_child_fungus", "overlay_male_wielded_corpse_mon_zombie_swimmer", "overlay_male_wielded_corpse_mon_zombie_mancroc", "overlay_male_wielded_corpse_mon_zombie_soldier", "overlay_male_wielded_corpse_mon_zombie_skull", "overlay_male_wielded_corpse_mon_zombie_brainless", "overlay_male_wielded_corpse_mon_zombie_survivor", "overlay_male_wielded_corpse_mon_zombie_brute_shocker", "overlay_male_wielded_corpse_mon_zombie_soldier_acid_1", "overlay_male_wielded_corpse_mon_zombie_soldier_blackops_2", "overlay_male_wielded_corpse_mon_zombie_soldier_blackops_1", "overlay_male_wielded_corpse_mon_zombie_soldier_acid_2", "overlay_male_wielded_corpse_mon_zombie_shriekling", "overlay_male_wielded_corpse_mon_zombie_ears", "overlay_male_wielded_corpse_mon_zombie_nullfield", "overlay_male_wielded_corpse_mon_zombie_waif", "overlay_male_wielded_corpse_mon_zombie_sproglodyte", "overlay_male_wielded_corpse_mon_zombie_creepy", "overlay_male_wielded_corpse_mon_zombie_anklebiter", "overlay_male_wielded_corpse_mon_zombie_bio_op", "overlay_male_wielded_corpse_mon_zombie_armored", "overlay_male_wielded_corpse_mon_zombie_prisoner", "overlay_male_wielded_corpse_mon_zombie_military_pilot"], "fg": 2000}, {"id": ["overlay_female_wielded_corpse_mon_zombie", "overlay_female_wielded_corpse_mon_zombie_grappler", "overlay_female_wielded_corpse_mon_zombie_biter", "overlay_female_wielded_corpse_mon_zombie_tough", "overlay_female_wielded_corpse_mon_zombie_hunter", "overlay_female_wielded_corpse_mon_zombie_brute", "overlay_female_wielded_corpse_mon_zombie_predator", "overlay_female_wielded_corpse_mon_zombie_necro", "overlay_female_wielded_corpse_mon_zombie_thorny", "overlay_female_wielded_corpse_mon_zombie_smoker", "overlay_female_wielded_corpse_mon_zombie_shady", "overlay_female_wielded_corpse_mon_zombie_master", "overlay_female_wielded_corpse_mon_zombie_acidic", "overlay_female_wielded_corpse_mon_zombie_fat", "overlay_female_wielded_corpse_mon_zombie_corrosive", "overlay_female_wielded_corpse_mon_zombie_screecher", "overlay_female_wielded_corpse_mon_zombie_scientist", "overlay_female_wielded_corpse_mon_zombie_runner", "overlay_female_wielded_corpse_mon_zombie_spitter", "overlay_female_wielded_corpse_mon_zombie_labsecurity", "overlay_female_wielded_corpse_mon_zombie_child", "overlay_female_wielded_corpse_mon_zombie_hulk", "overlay_female_wielded_corpse_mon_zombie_brute_grappler", "overlay_female_wielded_corpse_mon_zombie_brute_ninja", "overlay_female_wielded_corpse_mon_zombie_kevlar_2", "overlay_female_wielded_corpse_mon_zombie_kevlar_1", "overlay_female_wielded_corpse_mon_zombie_hazmat", "overlay_female_wielded_corpse_mon_zombie_electric", "overlay_female_wielded_corpse_mon_zombie_technician", "overlay_female_wielded_corpse_mon_zombie_fungus", "overlay_female_wielded_corpse_mon_zombie_cop", "overlay_female_wielded_corpse_mon_zombie_child_fungus", "overlay_female_wielded_corpse_mon_zombie_swimmer", "overlay_female_wielded_corpse_mon_zombie_mancroc", "overlay_female_wielded_corpse_mon_zombie_soldier", "overlay_female_wielded_corpse_mon_zombie_skull", "overlay_female_wielded_corpse_mon_zombie_brainless", "overlay_female_wielded_corpse_mon_zombie_survivor", "overlay_female_wielded_corpse_mon_zombie_brute_shocker", "overlay_female_wielded_corpse_mon_zombie_soldier_acid_1", "overlay_female_wielded_corpse_mon_zombie_soldier_blackops_2", "overlay_female_wielded_corpse_mon_zombie_soldier_blackops_1", "overlay_female_wielded_corpse_mon_zombie_soldier_acid_2", "overlay_female_wielded_corpse_mon_zombie_shriekling", "overlay_female_wielded_corpse_mon_zombie_ears", "overlay_female_wielded_corpse_mon_zombie_nullfield", "overlay_female_wielded_corpse_mon_zombie_waif", "overlay_female_wielded_corpse_mon_zombie_sproglodyte", "overlay_female_wielded_corpse_mon_zombie_creepy", "overlay_female_wielded_corpse_mon_zombie_anklebiter", "overlay_female_wielded_corpse_mon_zombie_bio_op", "overlay_female_wielded_corpse_mon_zombie_armored", "overlay_female_wielded_corpse_mon_zombie_prisoner", "overlay_female_wielded_corpse_mon_zombie_military_pilot"], "fg": 2001}, {"id": "overlay_female_wielded_bat_metal", "fg": 2005}, {"id": "overlay_male_wielded_bat_metal", "fg": 2004}, {"id": "overlay_wielded_mop", "fg": 2006}, {"id": "overlay_wielded_crowbar", "fg": 2008}, {"id": "overlay_male_wielded_screwdriver", "fg": 2010}, {"id": "overlay_female_wielded_screwdriver", "fg": 2007}, {"id": "overlay_wielded_shovel", "fg": 2012}, {"id": "overlay_wielded_wrench", "fg": 2009}, {"id": "overlay_wielded_hammer", "fg": 2011}, {"id": "overlay_male_wielded_pointy_stick", "fg": 2019}, {"id": "overlay_female_wielded_pointy_stick", "fg": 2025}, {"id": "overlay_male_wielded_spear_wood", "fg": 2020}, {"id": "overlay_female_wielded_spear_wood", "fg": 2023}, {"id": "overlay_male_wielded_spear_spike", "fg": 2013}, {"id": "overlay_female_wielded_spear_spike", "fg": 2014}, {"id": "overlay_male_wielded_spear_knife", "fg": 2018}, {"id": "overlay_female_wielded_spear_knife", "fg": 2026}, {"id": "overlay_male_wielded_spear_knife_superior", "fg": 2015}, {"id": "overlay_female_wielded_spear_knife_superior", "fg": 2021}, {"id": "overlay_male_wielded_spear_rebar", "fg": 2030}, {"id": "overlay_female_wielded_spear_rebar", "fg": 2017}, {"id": "overlay_male_wielded_spear_pipe", "fg": 2027}, {"id": "overlay_female_wielded_spear_pipe", "fg": 2028}, {"id": "overlay_male_wielded_spear_steel", "fg": 2029}, {"id": "overlay_female_wielded_spear_steel", "fg": 2022}, {"id": "overlay_male_wielded_spear_copper", "fg": 2024}, {"id": "overlay_female_wielded_spear_copper", "fg": 2016}, {"id": "overlay_male_wielded_2x4", "fg": 2032}, {"id": "overlay_female_wielded_2x4", "fg": 2031}, {"id": "overlay_wielded_scissors", "fg": 2033}, {"id": "overlay_wielded_bag_canvas", "fg": 2034}, {"id": "overlay_male_wielded_wood_sheet", "fg": 2036}, {"id": "overlay_female_wielded_wood_sheet", "fg": 2035}, {"id": "overlay_female_wielded_rag", "fg": 2038}, {"id": "overlay_male_wielded_rag", "fg": 2037}, {"id": "overlay_female_wielded_television", "fg": 2040}, {"id": "overlay_male_wielded_television", "fg": 2039}, {"id": "overlay_wielded_pipe", "fg": 2041}, {"id": "overlay_wielded_bottle_plastic", "fg": 2042}, {"id": "overlay_female_wielded_primitive_hammer", "fg": 2043}, {"id": "overlay_male_wielded_primitive_hammer", "fg": 2044}, {"id": "overlay_wielded_jar_glass", "fg": 2045}, {"id": "overlay_wielded_bottle_glass", "fg": 2046}, {"id": "overlay_wielded_box_medium", "fg": 2047}, {"id": "overlay_wielded_flashlight", "fg": 2049}, {"id": "overlay_wielded_heavy_flashlight", "fg": 2048}, {"id": "overlay_wielded_box_small", "fg": 2050}, {"id": "overlay_wielded_rock", "fg": 2051}, {"id": "overlay_male_wielded_splinter", "fg": 2052}, {"id": "overlay_female_wielded_splinter", "fg": 2053}, {"id": "overlay_wielded_ar15", "fg": 2054}, {"id": "overlay_female_wielded_chainsaw_off", "fg": 2056}, {"id": "overlay_male_wielded_chainsaw_off", "fg": 2055}, {"id": "overlay_wielded_makeshift_crowbar", "fg": 2057}, {"id": "overlay_wielded_brick", "fg": 2058}, {"id": "overlay_wielded_arming_sword", "fg": 2059}, {"id": "overlay_female_wielded_rebar", "fg": []}, {"id": "overlay_male_wielded_rebar", "fg": []}, {"id": "overlay_male_wielded_pot", "fg": 2061}, {"id": "overlay_female_wielded_pot", "fg": 2062}, {"id": "overlay_male_wielded_wood_panel", "fg": 2064}, {"id": "overlay_female_wielded_wood_panel", "fg": 2063}, {"id": "overlay_female_wielded_bwirebat", "fg": 2066}, {"id": "overlay_male_wielded_bwirebat", "fg": 2065}, {"id": "overlay_wielded_bag_plastic", "fg": 2067}, {"id": "overlay_female_wielded_bat", "fg": 2069}, {"id": "overlay_male_wielded_bat", "fg": 2068}, {"id": "overlay_wielded_rolling_pin", "fg": 2070}, {"id": "overlay_wielded_jar_3l_glass_sealed", "fg": 2071}, {"id": "overlay_wielded_sw_619", "fg": 2072}, {"id": "overlay_female_wielded_1st_aid", "fg": 2073}, {"id": "overlay_male_wielded_1st_aid", "fg": 2074}, {"id": "overlay_female_wielded_nailbat", "fg": 2076}, {"id": "overlay_male_wielded_nailbat", "fg": 2075}, {"id": "overlay_male_wielded_pillow", "fg": 2078}, {"id": "overlay_female_wielded_pillow", "fg": 2077}, {"id": "overlay_male_wielded_down_pillow", "fg": 2080}, {"id": "overlay_female_wielded_down_pillow", "fg": 2079}, {"id": "overlay_female_wielded_stick_long", "fg": 2081}, {"id": "overlay_male_wielded_stick_long", "fg": 2082}, {"id": "overlay_wielded_broom", "fg": 2083}, {"id": "overlay_female_wielded_ak74", "fg": 2085}, {"id": "overlay_male_wielded_ak74", "fg": 2084}, {"id": "overlay_wielded_antibiotics", "fg": 2087}, {"id": "overlay_wielded_weak_antibiotic", "fg": 2095}, {"id": "overlay_wielded_strong_antibiotic", "fg": 2096}, {"id": "overlay_wielded_calcium_tablet", "fg": 2098}, {"id": "overlay_wielded_vitamins", "fg": 2097}, {"id": "overlay_wielded_gummy_vitamins", "fg": 2086}, {"id": "overlay_wielded_antifungal", "fg": 2090}, {"id": "overlay_wielded_antiparasitic", "fg": 2092}, {"id": "overlay_wielded_iodine", "fg": 2093}, {"id": "overlay_wielded_prussian_blue", "fg": 2088}, {"id": "overlay_wielded_tramadol", "fg": 2094}, {"id": "overlay_wielded_codeine", "fg": 2089}, {"id": "overlay_wielded_oxycodone", "fg": 2091}, {"id": "overlay_wielded_steel_lump", "fg": 2099}, {"id": "overlay_male_wielded_teapot", "fg": 2101}, {"id": "overlay_female_wielded_teapot", "fg": 2100}, {"id": "overlay_wielded_steel_chunk", "fg": 2102}, {"id": "overlay_wielded_bottle_plastic_small", "fg": 2103}, {"id": "overlay_wielded_jar_3l_glass", "fg": 2104}, {"id": "overlay_female_wielded_stick", "fg": 2106}, {"id": "overlay_male_wielded_stick", "fg": 2105}, {"id": "overlay_wielded_log", "fg": 2107}, {"id": "overlay_male_wielded_rifle_flintlock", "fg": 2108}, {"id": "overlay_female_wielded_rifle_flintlock", "fg": 2109}, {"id": "overlay_wielded_sharp_rock", "fg": 2110}, {"id": "overlay_wielded_coffeemaker", "fg": 2111}, {"id": "overlay_wielded_box_large", "fg": 2112}, {"id": "overlay_wielded_fire_ax", "fg": 2114}, {"id": "overlay_wielded_ax", "fg": 2115}, {"id": "overlay_wielded_hatchet", "fg": 2113}, {"id": "overlay_wielded_katana", "fg": 2116}, {"id": "overlay_female_wielded_pot_copper", "fg": 2118}, {"id": "overlay_male_wielded_pot_copper", "fg": 2117}, {"id": "overlay_female_wielded_corpse_generic_human", "fg": 2119}, {"id": "overlay_male_wielded_corpse_generic_human", "fg": 2120}, {"id": "overlay_female_wielded_nailboard", "fg": 2121}, {"id": "overlay_male_wielded_nailboard", "fg": 2122}, {"id": "overlay_male_mutation_SKIN_TAN", "fg": 2124}, {"id": "overlay_female_mutation_SKIN_TAN", "fg": 2123}, {"id": "overlay_female_mutation_SKIN_LIGHT", "fg": 2125}, {"id": "overlay_male_mutation_SKIN_LIGHT", "fg": 2126}, {"id": "overlay_female_mutation_SKIN_PINK", "fg": 2128}, {"id": "overlay_male_mutation_SKIN_PINK", "fg": 2127}, {"id": "overlay_female_mutation_SKIN_DARK", "fg": 2130}, {"id": "overlay_male_mutation_SKIN_DARK", "fg": 2129}, {"id": "overlay_female_mutation_SKIN_MEDIUM", "fg": 2131}, {"id": "overlay_male_mutation_SKIN_MEDIUM", "fg": 2132}], "//": "range 1585 to 2144", "sprite_width": 32, "sprite_height": 64, "sprite_offset_x": 0, "sprite_offset_y": -32}, {"file": "large.png", "tiles": [{"id": "t_tree_young", "fg": [{"weight": 100, "sprite": 2149}, {"weight": 100, "sprite": 2155}], "bg": 2145}, {"id": "t_tree_young_season_summer", "fg": [{"weight": 100, "sprite": 2151}, {"weight": 100, "sprite": 2154}], "bg": 2148, "rotates": false}, {"id": "t_tree_young_season_autumn", "fg": [{"weight": 100, "sprite": 2156}, {"weight": 100, "sprite": 2152}], "bg": 2147, "rotates": false}, {"id": "t_tree_young_season_winter", "fg": [{"weight": 100, "sprite": 2150}, {"weight": 100, "sprite": 2153}], "bg": 2146, "rotates": false}, {"id": "f_fridge", "fg": 2157}, {"id": "f_rotary_clothesline", "fg": 2158}, {"id": "mon_blob_small", "fg": [{"weight": 1, "sprite": 2160}, {"weight": 1, "sprite": 2159}]}, {"id": "mon_ant_soldier", "fg": 2162}, {"id": "mon_ant_acid_soldier", "fg": 2166}, {"id": "corpse_mon_ant_soldier", "fg": 2163}, {"id": "corpse_mon_ant_acid_soldier", "fg": 2167}, {"id": "mon_ant_queen", "fg": 2168}, {"id": "mon_ant_acid_queen", "fg": 2165}, {"id": "corpse_mon_ant_queen", "fg": 2164}, {"id": "corpse_mon_ant_acid_queen", "fg": 2161}, {"id": "mon_zombie_biter", "fg": 2169}, {"id": "mon_zombie_crawler", "fg": 2170}, {"id": "mon_blob", "fg": 2171}, {"id": "mon_boomer", "fg": 2172}, {"id": "mon_zombie", "fg": [{"weight": 100, "sprite": 2177}, {"weight": 150, "sprite": 2173}, {"weight": 100, "sprite": 2178}, {"weight": 100, "sprite": 2176}, {"weight": 150, "sprite": 2175}]}, {"id": ["corpse_mon_zombie", "corpse_mon_zombie_grappler", "corpse_mon_zombie_biter", "corpse_mon_zombie_hunter", "corpse_mon_zombie_brute", "corpse_mon_zombie_predator", "corpse_mon_zombie_necro", "corpse_mon_zombie_thorny", "corpse_mon_zombie_smoker", "corpse_mon_zombie_shady", "corpse_mon_zombie_master", "corpse_mon_zombie_acidic", "corpse_mon_zombie_fat", "corpse_mon_zombie_corrosive", "corpse_mon_zombie_screecher", "corpse_mon_zombie_scientist", "corpse_mon_zombie_runner", "corpse_mon_zombie_labsecurity", "corpse_mon_zombie_child", "corpse_mon_zombie_hulk", "corpse_mon_zombie_brute_grappler", "corpse_mon_zombie_brute_ninja", "corpse_mon_zombie_kevlar_2", "corpse_mon_zombie_kevlar_1", "corpse_mon_zombie_hazmat", "corpse_mon_zombie_electric", "corpse_mon_zombie_technician", "corpse_mon_zombie_fungus", "corpse_mon_zombie_cop", "corpse_mon_zombie_child_fungus", "corpse_mon_zombie_swimmer", "corpse_mon_zombie_mancroc", "corpse_mon_zombie_soldier", "corpse_mon_zombie_skull", "corpse_mon_zombie_brainless", "corpse_mon_zombie_survivor", "corpse_mon_zombie_brute_shocker", "corpse_mon_zombie_soldier_acid_1", "corpse_mon_zombie_soldier_blackops_2", "corpse_mon_zombie_soldier_blackops_1", "corpse_mon_zombie_soldier_acid_2", "corpse_mon_zombie_shriekling", "corpse_mon_zombie_ears", "corpse_mon_zombie_nullfield", "corpse_mon_zombie_waif", "corpse_mon_zombie_sproglodyte", "corpse_mon_zombie_creepy", "corpse_mon_zombie_anklebiter", "corpse_mon_zombie_bio_op", "corpse_mon_zombie_armored", "corpse_mon_zombie_prisoner", "corpse_mon_zombie_military_pilot"], "fg": 2174}, {"id": "mon_blob_large", "fg": 2179}, {"id": "mon_zombie_hunter", "fg": 2180}, {"id": "mon_zombear", "fg": 2181}, {"id": "mon_moose", "fg": [{"weight": 2, "sprite": 2182}, {"weight": 2, "sprite": 2183}]}, {"id": "mon_zombie_electric", "fg": [{"weight": 1, "sprite": 2186}, {"weight": 1, "sprite": 2187}]}, {"id": "mon_zombie_nullfield", "fg": [{"weight": 1, "sprite": 2185}, {"weight": 1, "sprite": 2184}]}, {"id": "mon_zoose", "fg": 2188}, {"id": "mon_zombie_fat", "fg": [{"weight": 1, "sprite": 2190}, {"weight": 1, "sprite": 2189}]}, {"id": "mon_wasp", "fg": 2191}, {"id": "mon_zombie_cop", "fg": 2192}, {"id": "mon_zombie_labsecurity", "fg": 2193}, {"id": "mon_zombie_tough", "fg": 2195}, {"id": "corpse_mon_zombie_tough", "fg": 2194}, {"id": "mon_zombie_acidic", "fg": 2197}, {"id": "mon_zombie_corrosive", "fg": 2196}, {"id": "mon_zombie_gasbag", "fg": [{"weight": 1, "sprite": 2199}, {"weight": 1, "sprite": 2198}]}, {"id": "mon_zombie_spitter", "fg": 2200}, {"id": "corpse_mon_zombie_spitter", "fg": 2201}, {"id": "mon_zombie_grabber", "fg": 2202}, {"id": "mon_zombie_grappler", "fg": 2203}, {"id": "mon_zombie_smoker", "fg": 2204}, {"id": "mon_zombie_brute", "fg": [{"weight": 100, "sprite": 2205}, {"weight": 100, "sprite": 2206}, {"weight": 50, "sprite": 2207}, {"weight": 50, "sprite": 2208}]}, {"id": "mon_cat", "fg": [{"weight": 1, "sprite": 2209}, {"weight": 1, "sprite": 2210}]}, {"id": "mon_bear", "fg": 2211}, {"id": "mon_mi_go", "fg": 2212}, {"id": "mon_zombie_necro", "fg": 2213}, {"id": "mon_zombie_shady", "fg": 2214}, {"id": "mon_zombie_runner", "fg": 2215}, {"id": "mon_zombie_scientist", "fg": [{"weight": 1, "sprite": 2216}, {"weight": 1, "sprite": 2217}]}, {"id": "mon_zombie_rot", "fg": 2218}], "//": "range 2145 to 2224", "sprite_width": 64, "sprite_height": 64, "sprite_offset_x": -16, "sprite_offset_y": -32}, {"file": "huge.png", "tiles": [{"id": "mon_zombie_hulk", "fg": 2225}], "//": "range 2225 to 2240", "sprite_width": 64, "sprite_height": 96, "sprite_offset_x": -16, "sprite_offset_y": -64}, {"file": "giant.png", "tiles": [{"id": "t_tree_dead", "fg": [{"weight": 100, "sprite": 2242}, {"weight": 100, "sprite": 2243}, {"weight": 100, "sprite": 2244}, {"weight": 100, "sprite": 2241}, {"weight": 100, "sprite": 2613}], "bg": 2266}, {"id": "t_tree_dead_season_summer", "fg": [{"weight": 100, "sprite": 2242}, {"weight": 100, "sprite": 2243}, {"weight": 100, "sprite": 2244}, {"weight": 100, "sprite": 2241}, {"weight": 100, "sprite": 2613}], "bg": 2265}, {"id": "t_tree_dead_season_autumn", "fg": [{"weight": 100, "sprite": 2242}, {"weight": 100, "sprite": 2243}, {"weight": 100, "sprite": 2244}, {"weight": 100, "sprite": 2241}, {"weight": 100, "sprite": 2613}], "bg": 2264}, {"id": "t_tree_dead_season_winter", "fg": [{"weight": 100, "sprite": 2242}, {"weight": 100, "sprite": 2243}, {"weight": 100, "sprite": 2244}, {"weight": 100, "sprite": 2241}, {"weight": 100, "sprite": 2613}], "bg": 2263}, {"id": "t_tree_birch", "fg": [{"weight": 1, "sprite": 2245}, {"weight": 1, "sprite": 2247}], "bg": 2266}, {"id": "t_tree_birch_season_summer", "fg": [{"weight": 1, "sprite": 2245}, {"weight": 1, "sprite": 2247}], "bg": 2265}, {"id": "t_tree_birch_season_winter", "fg": [{"weight": 1, "sprite": 2250}, {"weight": 1, "sprite": 2248}], "bg": 2263}, {"id": "t_tree_birch_season_autumn", "fg": [{"weight": 1, "sprite": 2249}, {"weight": 1, "sprite": 2246}], "bg": 2264}, {"id": "t_tree", "fg": [{"weight": 100, "sprite": 2256}, {"weight": 100, "sprite": 2261}, {"weight": 100, "sprite": 2259}, {"weight": 100, "sprite": 2257}], "bg": 2266}, {"id": "t_tree_season_summer", "fg": [{"weight": 100, "sprite": 2260}, {"weight": 100, "sprite": 2252}, {"weight": 100, "sprite": 2254}, {"weight": 100, "sprite": 2258}], "bg": 2265}, {"id": "t_tree_season_autumn", "fg": [{"weight": 100, "sprite": 2262}, {"weight": 100, "sprite": 2251}, {"weight": 100, "sprite": 2255}, {"weight": 100, "sprite": 2253}], "bg": 2264}, {"id": "t_tree_season_winter", "fg": [{"weight": 100, "sprite": 2242}, {"weight": 100, "sprite": 2243}, {"weight": 100, "sprite": 2244}, {"weight": 100, "sprite": 2241}], "bg": 2263}], "//": "range 2241 to 2272", "sprite_width": 96, "sprite_height": 96, "sprite_offset_x": -32, "sprite_offset_y": -64}, {"file": "incomplete.png", "tiles": [{"id": "t_strconc_wall", "fg": 2273}, {"id": "t_splitrail_fence_season_spring", "fg": 2274, "bg": 938}, {"id": "t_splitrail_fence_season_summer", "fg": 2274, "bg": 937}, {"id": "t_splitrail_fence_season_autumn", "fg": 2274, "bg": 946}, {"id": "t_splitrail_fence_season_winter", "fg": 2274, "bg": 587}, {"id": ["t_window_boarded", "t_window_boarded_noglass"], "fg": 2275, "bg": 333}, {"id": "t_door_metal_o", "fg": 2276}, {"id": "t_chainfence", "multitile": true, "rotates": false, "fg": 2281, "bg": 417, "additional_tiles": [{"id": "edge", "bg": 417, "fg": [2280, 2281]}, {"id": "end_piece", "bg": 417, "fg": [2280, 2281, 2280, 2281]}, {"bg": 417, "id": "unconnected", "fg": 2281}]}, {"id": "t_chaingate_c", "fg": 2278, "bg": 417}, {"id": "t_chaingate_l", "fg": 2279, "bg": 417}, {"id": "t_chaingate_o", "fg": 2282, "bg": 417}, {"id": "t_chainfence_posts", "fg": 2277, "bg": 417}, {"id": "t_door_metal_c", "fg": 2283}, {"id": "t_pavement_y", "fg": 2284}, {"id": "t_pavement_y_season_winter", "fg": 587}, {"id": "t_console_broken", "fg": 2285}, {"id": "t_window_frame", "fg": 2286}, {"id": "t_wall_metal", "fg": 2287}, {"id": "t_dirtmound", "fg": 2288}, {"id": "t_railing", "fg": 2289, "bg": 938}, {"id": "t_console", "fg": 2290}, {"id": ["t_window_reinforced", "t_window_reinforced_noglass"], "fg": 2291, "bg": 333}, {"id": "t_underbrush", "fg": [{"weight": 100, "sprite": 2292}, {"weight": 100, "sprite": 2294}], "bg": 938}, {"id": "t_underbrush_harvested", "fg": 2293, "bg": 938}, {"id": "t_underbrush_season_summer", "fg": [{"weight": 100, "sprite": 2292}, {"weight": 100, "sprite": 2294}], "bg": 937}, {"id": "t_underbrush_harvested_season_summer", "fg": 2293, "bg": 937}, {"id": "t_underbrush_season_autumn", "fg": [{"weight": 100, "sprite": 2292}, {"weight": 100, "sprite": 2294}], "bg": 946}, {"id": "t_underbrush_harvested_season_winter", "fg": 2293, "bg": 586}, {"id": "t_underbrush_season_winter", "fg": [{"weight": 100, "sprite": 2292}, {"weight": 100, "sprite": 2294}], "bg": 586}, {"id": "t_trunk", "multitile": true, "fg": [2295, 2296], "bg": [{"weight": 100, "sprite": 938}, {"weight": 100, "sprite": 943}]}, {"id": "t_trunk_season_summer", "multitile": true, "fg": [2295, 2296], "bg": [{"weight": 100, "sprite": 937}, {"weight": 100, "sprite": 941}]}, {"id": "t_trunk_season_autumn", "multitile": true, "fg": [2295, 2296], "bg": [{"weight": 100, "sprite": 946}, {"weight": 100, "sprite": 948}]}, {"id": "t_trunk_season_winter", "multitile": true, "fg": [2295, 2296], "bg": 587}, {"id": "f_sink", "multitile": true, "fg": 2305, "additional_tiles": [{"id": "center", "fg": 2300}, {"id": "corner", "fg": [2304, 2310, 2298, 2306]}, {"id": "t_connection", "fg": [2307, 2311, 2297, 2299]}, {"id": "edge", "fg": [2302, 2303]}, {"id": "end_piece", "fg": [2309, 2308, 2312, 2301]}, {"id": "unconnected", "fg": 2305}]}, {"id": "fd_acid", "fg": 2313}, {"id": "fd_blood", "fg": 2314}, {"id": ["fd_blood_insect", "fd_blood_invertebrate"], "fg": 2315}, {"id": "fd_web", "fg": [{"weight": 100, "sprite": 2319}, {"weight": 100, "sprite": 2316}, {"weight": 25, "sprite": 2318}, {"weight": 25, "sprite": 2317}]}, {"id": "fd_smoke", "fg": 2321}, {"id": "fd_fungal_haze", "fg": 2322}, {"id": "fd_nuke_gas", "fg": 2320}, {"id": "fd_electricity", "fg": [{"weight": 100, "sprite": 2324}, {"weight": 100, "sprite": 2323}]}, {"id": "knife_steak", "fg": 2325}, {"id": "soap", "fg": 2326}, {"id": "condom", "fg": 2327}, {"id": "hk_mp5", "fg": 2328}, {"id": "mag_pistol", "fg": 2329}, {"id": "pitchfork", "fg": 2330}, {"id": "rope_6", "fg": 2331}, {"id": "mosin91_30", "fg": 2332}, {"id": "powder_candy", "fg": 2333}, {"id": "ref_lighter", "fg": 2334}, {"id": "extinguisher", "fg": 2335}, {"id": "waffleiron", "fg": 2336}, {"id": "wood_beam", "fg": 2337}, {"id": "brazier", "fg": 2338}, {"id": "pliers", "fg": 2339}, {"id": "soldering_iron", "fg": 2340}, {"id": "stanag30", "fg": 2341}, {"id": "charcoal", "fg": 2342}, {"id": "acoustic_guitar", "fg": 2343}, {"id": "colt_army", "fg": 2344}, {"id": "keg", "fg": 2345}, {"id": "toolbox", "fg": 2346}, {"id": "bubblewrap", "fg": 2347}, {"id": "spray_can", "fg": 2348}, {"id": "stepladder", "fg": 2349}, {"id": "smart_phone", "fg": 2350}, {"id": "duct_tape", "fg": 2351}, {"id": "colt_lightning", "fg": 2352}, {"id": "colt_navy", "fg": 2353}, {"id": "mag_smg", "fg": 2354}, {"id": "sw_610", "fg": 2355}, {"id": "hotplate", "fg": 2356}], "//": "range 2273 to 2368"}, {"file": "fillerhoder.png", "tiles": [{"id": "vp_kitchen_unit", "fg": 2582, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_stowboard_horizontal", "fg": 2482, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2455}]}, {"id": "vp_woodboard_ne", "fg": 2443, "rotates": true}, {"id": "vp_light_red", "bg": [], "fg": 2519, "rotates": true}, {"id": "vp_board_horizontal", "fg": 2506, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2417}]}, {"id": "vp_board_sw", "fg": 2480, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2372}]}, {"id": "vp_hdboard_horizontal", "fg": 2401, "rotates": true}, {"id": "vp_gas_tank", "fg": 2560, "rotates": true}, {"id": "vp_wheel_wheelchair", "fg": 2544, "rotates": true}, {"id": "[vp_xlframe_cross],[vp_fxlframe_cross]", "fg": 2515, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2521}]}, {"id": "vp_reinforced_solar_panel", "fg": [], "bg": 2464, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_storage_truck", "fg": 2436, "rotates": true}, {"id": "vp_wheel_motorbike_steerable", "fg": 2439, "rotates": true}, {"id": "vp_hddoor_sliding", "fg": 2418, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_wheel_unicycle", "fg": 2544, "rotates": true}, {"id": "vp_folding_seat", "fg": 2434, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": ["vp_omnicam", "vp_omnomnicam"], "fg": 2470, "bg": 2532, "rotates": true}, {"id": "vp_reclining_seat", "fg": 2434, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_roof_cloth", "fg": [], "rotates": true}, {"id": "vp_door_internal", "fg": 2535, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2507}, {"id": "broken", "fg": 2487}]}, {"id": "vp_hdboard_nw", "fg": 2588, "rotates": true}, {"id": "vp_hdboard_ne", "fg": 2431, "rotates": true}, {"id": "vp_plating_hard", "fg": 2553, "rotates": true}, {"id": "vp_stowboard_ne", "fg": 2427, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2423}]}, {"id": "vp_light_blue", "bg": [], "fg": 2519, "rotates": true}, {"id": "vp_wheel_armor", "fg": 2503, "rotates": true}, {"id": "vp_plating_steel", "fg": 2525, "rotates": true}, {"id": "[vp_xlhalfboard_se],[vp_fxlhalfboard_se]", "fg": 2517, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2522}]}, {"id": "vp_muffler", "fg": 2578, "rotates": true}, {"id": "vp_wheel_wide_steerable", "fg": 2543, "rotates": true}, {"id": "vp_frame_wood_handle", "fg": 2537, "rotates": true}, {"id": "vp_lit_aisle_horizontal", "fg": 2512, "rotates": true}, {"id": "vp_water_tank", "fg": 2369, "rotates": true}, {"id": "vp_plating_superalloy", "fg": 2551, "rotates": true}, {"id": "[vp_xlframe_horizontal_2],[vp_fxlframe_horizontal_2]", "fg": 2571, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2565}]}, {"id": "vp_board_vertical", "fg": 2370, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2563}]}, {"id": "vp_mounted_browning", "fg": 2388, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_frame_wood_cover", "fg": 2558, "rotates": true}, {"id": "vp_blade_vertical", "fg": 2523, "rotates": true}, {"id": "vp_frame_wood_ne", "fg": 2529, "rotates": true}, {"id": "vp_hdhatch", "fg": 2441, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_solar_panel_v2", "fg": 2464, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_saddle", "fg": 2490, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_solar_panel_v3", "fg": 2464, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_hdhalfboard_horizontal", "fg": 2413, "rotates": true}, {"id": "vp_frame_sw", "fg": 2391, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2548}]}, {"id": "vp_foot_pedals", "fg": 2438, "rotates": true}, {"id": "vp_engine_electric_large", "fg": 2419, "rotates": true}, {"id": "vp_hdframe_se", "fg": 2473, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2471}]}, {"id": "vp_hdhalfboard_vertical", "fg": 2446, "rotates": true}, {"id": "vp_woodhalfboard_nw", "fg": 2513, "rotates": true}, {"id": "vp_box", "fg": 2408, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_spike", "fg": 2387, "rotates": true}, {"id": "[vp_xlhalfboard_sw],[vp_fxlhalfboard_sw", "fg": 2554, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2579}]}, {"id": "vp_wheel", "fg": 2503, "rotates": true}, {"id": "vp_hdhalfboard_horizontal_2", "fg": 2466, "rotates": true}, {"id": "vp_frame_se", "fg": 2528, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2516}]}, {"id": "vp_chemlab", "fg": 2582, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_hdhalfboard_sw", "fg": 2566, "rotates": true}, {"id": "vp_wheel_small", "fg": 2587, "rotates": true}, {"id": "vp_hdhalfboard_vertical_2", "fg": 2532, "rotates": true}, {"id": "vp_controls", "fg": 2502, "rotates": true}, {"id": "vp_door_shutter", "fg": 2590, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2507}, {"id": "broken", "fg": 2487}]}, {"id": "vp_frame_horizontal", "fg": 2505, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2580}]}, {"id": "vp_hddoor_trunk", "fg": 2524, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_hdframe_horizontal_2", "fg": 2398, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2530}]}, {"id": "vp_engine_1cyl", "fg": 2531, "rotates": true}, {"id": "vp_frame_ne", "fg": 2474, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2550}]}, {"id": "vp_hdstowboard_sw", "fg": 2555, "rotates": true}, {"id": "[vp_xlframe_se],[vp_fxlframe_se]", "fg": 2449, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2486}]}, {"id": "vp_frame_horizontal_2", "fg": 2591, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2374}]}, {"id": "vp_floodlight", "fg": 2520, "bg": 2562, "rotates": true}, {"id": "vp_woodhalfboard_sw", "fg": 2501, "rotates": true}, {"id": "vp_hdboard_sw", "fg": 2412, "rotates": true}, {"id": "vp_frame_handle", "fg": 2537, "rotates": true}, {"id": "vp_engine_inline4", "fg": 2469, "rotates": true}, {"id": "vp_fusion_gun", "fg": 2450, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_small_storage_battery", "fg": 2436, "rotates": true}, {"id": "vp_engine_plasma", "fg": 2376, "rotates": true}, {"id": "vp_hdframe_cover", "fg": 2484, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2564}]}, {"id": "vp_frame_wood_horizontal_2", "fg": 2428, "rotates": true}, {"id": "vp_plating_military", "fg": 2525, "rotates": true}, {"id": "vp_turret_mount", "fg": 2453, "bg": 2532, "rotates": true}, {"id": "vp_basketsm", "fg": 2542, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_flamethrower", "fg": 2589, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_lit_aisle_vertical", "fg": 2540, "rotates": true}, {"id": "vp_battery_motorbike", "fg": 2436, "rotates": true}, {"id": "vp_stowboard_sw", "fg": 2400, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2477}]}, {"id": "vp_frame_nw", "fg": 2572, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2461}]}, {"id": "vp_hdstowboard_nw", "fg": 2510, "rotates": true}, {"id": "vp_woodboard_nw", "fg": 2513, "rotates": true}, {"id": "vp_hddoor_internal", "fg": 2375, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_engine_v8", "fg": 2534, "rotates": true}, {"id": "vp_frame_vertical", "fg": 2576, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2386}]}, {"id": "vp_seat", "fg": 2434, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_recharge_station", "fg": 2436, "bg": 2408, "rotates": true}, {"id": "vp_hdframe_vertical_2", "fg": 2495, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2567}]}, {"id": "vp_wheel_armor_steerable", "fg": 2503, "rotates": true}, {"id": "vp_plasma_gun", "fg": 2435, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "[vp_xlhalfboard_vertical_2],[vp_fxlhalfboard_vertical_2]", "fg": 2377, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2592}]}, {"id": "vp_hdframe_cross", "fg": 2406, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2583}]}, {"id": "vp_storage_battery", "fg": 2436, "rotates": true}, {"id": "vp_hdboard_vertical", "fg": 2492, "rotates": true}, {"id": "vp_engine_vtwin", "fg": 2437, "rotates": true}, {"id": "vp_frame_cover", "fg": 2496, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2432}]}, {"id": "vp_wheel_bicycle_steerable", "fg": 2544, "rotates": true}, {"id": "vp_hdframe_ne", "fg": 2549, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2460}]}, {"id": "vp_laser_gun", "fg": 2450, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_frame_wood_nw", "fg": 2410, "rotates": true}, {"id": "vp_washing_machine", "fg": [], "rotates": false}, {"id": "vp_stowboard_se", "fg": 2538, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2500}]}, {"id": "vp_board_ne", "fg": 2458, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2468}]}, {"id": "vp_aisle_horizontal", "fg": 2512, "rotates": true}, {"id": "vp_storage_car", "fg": 2436, "rotates": true}, {"id": "vp_stowboard_vertical", "fg": 2575, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2498}]}, {"id": "vp_hdframe_sw", "fg": 2596, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2493}]}, {"id": "vp_hdboard_se", "fg": 2494, "rotates": true}, {"id": "vp_wheel_wide", "fg": 2543, "rotates": true}, {"id": "vp_veh_forge", "fg": 2442, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_frame_wood_vertical", "fg": 2526, "rotates": true}, {"id": "vp_woodboard_horizontal", "fg": 2499, "rotates": true}, {"id": "vp_hatch_opaque", "fg": 2404, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2507}, {"id": "broken", "fg": 2487}]}, {"id": "vp_frame_wood_horizontal", "fg": 2444, "rotates": true}, {"id": "vp_frame_vertical_2", "fg": 2414, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2545}]}, {"id": "vp_hdstowboard_vertical", "fg": 2570, "rotates": true}, {"id": "vp_woodhalfboard_se", "fg": 2508, "rotates": true}, {"id": "vp_minifridge", "fg": [], "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_woodhalfboard_vertical", "fg": 2384, "rotates": true}, {"id": "vp_trunk", "fg": 2408, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_veh_table", "fg": [], "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_aisle_vertical", "fg": 2540, "rotates": true}, {"id": "vp_door_opaque", "fg": 2462, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2507}, {"id": "broken", "fg": 2487}]}, {"id": "vp_hatch", "fg": 2594, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2507}, {"id": "broken", "fg": 2487}]}, {"id": "vp_roof", "fg": 2562, "rotates": true}, {"id": "vp_woodhalfboard_vertical_2", "fg": 2384, "rotates": true}, {"id": "vp_trunk_floor", "fg": 2581, "rotates": true}, {"id": "vp_m249", "fg": 2388, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_external_gas_tank", "fg": 2560, "rotates": true}, {"id": "vp_woodboard_vertical", "fg": 2384, "rotates": true}, {"id": "vp_hdhatch_opaque", "fg": 2568, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_wheel_motorbike", "fg": 2439, "rotates": true}, {"id": "vp_engine_v12", "fg": 2422, "bg": 2534, "rotates": true}, {"id": "vp_hydrogen_tank", "fg": 2448, "rotates": true}, {"id": "vp_hddoor", "fg": 2593, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_hdstowboard_horizontal", "fg": 2393, "rotates": true}, {"id": "vp_woodhalfboard_horizontal", "fg": 2499, "rotates": true}, {"id": "vp_cargo_space", "fg": [], "bg": 2595, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_basketlg", "fg": 2542, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_hdstowboard_se", "fg": 2465, "rotates": true}, {"id": "vp_seatbelt", "fg": 2379, "rotates": true}, {"id": "vp_v_curtain", "fg": [], "rotates": true}, {"id": "vp_minireactor", "fg": 2390, "rotates": true}, {"id": "[vp_xlhalfboard_vertical],[vp_fxlhalfboard_vertical]", "fg": 2456, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2373}]}, {"id": "vp_hdstowboard_ne", "fg": 2476, "rotates": true}, {"id": "vp_wheel_small_steerable", "fg": 2587, "rotates": true}, {"id": "vp_wheel_caster", "fg": 2504, "rotates": true, "multitile": false}, {"id": "vp_woodboard_se", "fg": 2508, "rotates": true}, {"id": "vp_frame_wood_se", "fg": 2397, "rotates": true}, {"id": "vp_seatbelt_heavyduty", "fg": 2379, "rotates": true}, {"id": "vp_engine_v6", "fg": 2422, "rotates": true}, {"id": "[vp_xlhalfboard_ne],[vp_fxlhalfboard_ne]", "fg": 2451, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2559}]}, {"id": "[vp_xlframe_vertical],[vp_fxlframe_vertical]", "fg": 2481, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2420}]}, {"id": "[vp_xlframe_vertical_2],[vp_fxlframe_vertical_2]", "fg": 2497, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2407}]}, {"id": "vp_frame_cross", "fg": 2416, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2488}]}, {"id": "vp_reinforced_windshield", "fg": [], "bg": 2403, "rotates": true}, {"id": "[vp_xlframe_ne],[vp_fxlframe_ne]", "fg": 2447, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2547}]}, {"id": "vp_hdhalfboard_se", "fg": 2491, "rotates": true}, {"id": "vp_engine_electric", "fg": 2479, "rotates": true}, {"id": "vp_wheel_steerable", "fg": 2503, "rotates": true}, {"id": "vp_woodhalfboard_horizontal_2", "fg": 2499, "rotates": true}, {"id": "vp_frame_wood_sw", "fg": 2557, "rotates": true}, {"id": "vp_reinforced_solar_panel_v2", "fg": [], "bg": 2464, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_wing_mirror", "fg": 2454, "rotates": true}, {"id": "[vp_xlframe_horizontal],[vp_fxlframe_horizontal]", "fg": 2527, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2457}]}, {"id": "[vp_xlframe_nw],[vp_fxlframe_nw]", "fg": 2509, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2426}]}, {"id": "vp_mounted_mk19", "fg": 2388, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_plating_wood", "fg": [], "rotates": true}, {"id": "vp_board_nw", "fg": 2514, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2563}]}, {"id": "vp_bed", "fg": 2402, "rotates": true}, {"id": "vp_plating_spiked", "fg": 2396, "rotates": true}, {"id": "[vp_xlhalfboard_horizontal_2],[vp_fxlhalfboard_horizontal_2]", "fg": 2425, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2424}]}, {"id": "vp_gas_tank_small", "fg": 2560, "rotates": true}, {"id": "vp_hdframe_horizontal", "fg": 2552, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2561}]}, {"id": "vp_hddoor_opaque", "fg": 2405, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_craft_rig", "fg": 2582, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "[vp_xlhalfboard_horizontal],[vp_fxlhalfboard_horizontal]", "fg": 2573, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2378}]}, {"id": "vp_hddoor_shutter", "fg": 2429, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2585}, {"id": "broken", "fg": 2511}]}, {"id": "vp_frame_wood_cross", "fg": 2536, "rotates": true}, {"id": "vp_hdhalfboard_nw", "fg": 2478, "rotates": true}, {"id": "vp_blade_horizontal", "fg": 2518, "rotates": true}, {"id": "[vp_xlframe_sw],[vp_fxlframe_sw]", "fg": 2445, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2574}]}, {"id": "vp_woodboard_sw", "fg": 2501, "rotates": true}, {"id": "vp_welding_rig", "fg": 2421, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_frame_wood_vertical_2", "fg": 2595, "rotates": true}, {"id": "vp_solar_panel", "fg": 2464, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2487}]}, {"id": "vp_wheel_bicycle", "fg": 2544, "rotates": true}, {"id": "[vp_xlframe_cover],[vp_fxlframe_cover]", "fg": 2394, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2569}]}, {"id": "vp_hdroof", "fg": 2532, "rotates": true}, {"id": "vp_hdframe_vertical", "fg": 2380, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2409}]}, {"id": "vp_board_se", "fg": 2392, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2430}]}, {"id": "vp_woodhalfboard_ne", "fg": 2443, "rotates": true}, {"id": "vp_stowboard_nw", "fg": 2541, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2556}]}, {"id": "vp_door_sliding", "fg": 2584, "rotates": true, "multitile": true, "additional_tiles": [{"id": "open", "fg": 2507}, {"id": "broken", "fg": 2487}]}, {"id": "vp_hdframe_nw", "fg": 2483, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2472}]}, {"id": "vp_hdhalfboard_ne", "fg": 2382, "rotates": true}, {"id": "[vp_xlhalfboard_nw],[vp_fxlhalfboard_nw]", "fg": 2385, "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": 2389}]}], "//": "range 2369 to 2608"}, {"file": "opengameartgiant.png", "tiles": [{"id": "t_tree_chestnut", "fg": 2610, "bg": 2266}, {"id": "t_tree_chestnut_season_summer", "fg": 2613, "bg": 2265}, {"id": "t_tree_chestnut_season_autumn", "fg": 2613, "bg": 2264}, {"id": "t_tree_chestnut_season_winter", "fg": 2613, "bg": []}, {"id": "t_tree_pine", "fg": 2614, "bg": 2266}, {"id": "t_tree_pine_season_summer", "fg": 2614, "bg": 2265}, {"id": "t_tree_pine_season_autumn", "fg": 2614, "bg": 2264}, {"id": "t_tree_pine_season_winter", "fg": 2614, "bg": []}, {"id": "t_tree_deadpine", "fg": 2612, "bg": 2266}, {"id": "t_tree_deadpine_season_summer", "fg": 2612, "bg": 2265}, {"id": "t_tree_deadpine_season_autumn", "fg": 2612, "bg": 2264}, {"id": "t_tree_deadpine_season_winter", "fg": 2612, "bg": []}, {"id": "t_tree_hickory", "fg": 2611, "bg": 2266}, {"id": "t_tree_hickory_season_summer", "fg": 2611, "bg": 2265}, {"id": "t_tree_hickory_season_autumn", "fg": 2611, "bg": 2264}, {"id": "t_tree_hickory_season_winter", "fg": 2609, "bg": []}, {"id": "t_tree_hickory_dead", "fg": 2609, "bg": 2266}, {"id": "t_tree_hickory_dead_season_summer", "fg": 2609, "bg": 2265}, {"id": "t_tree_hickory_dead_season_autumn", "fg": 2609, "bg": 2264}, {"id": "t_tree_hickory_dead_season_winter", "fg": 2609, "bg": []}], "//": "range 2609 to 2624", "sprite_width": 96, "sprite_height": 96, "sprite_offset_x": -32, "sprite_offset_y": -64}, {"file": "fallback.png", "tiles": [], "ascii": [{"offset": 0, "bold": false, "color": "BLACK"}, {"offset": 256, "bold": true, "color": "WHITE"}, {"offset": 512, "bold": false, "color": "WHITE"}, {"offset": 768, "bold": true, "color": "BLACK"}, {"offset": 1024, "bold": false, "color": "RED"}, {"offset": 1280, "bold": false, "color": "GREEN"}, {"offset": 1536, "bold": false, "color": "BLUE"}, {"offset": 1792, "bold": false, "color": "CYAN"}, {"offset": 2048, "bold": false, "color": "MAGENTA"}, {"offset": 2304, "bold": false, "color": "YELLOW"}, {"offset": 2560, "bold": true, "color": "RED"}, {"offset": 2816, "bold": true, "color": "GREEN"}, {"offset": 3072, "bold": true, "color": "BLUE"}, {"offset": 3328, "bold": true, "color": "CYAN"}, {"offset": 3584, "bold": true, "color": "MAGENTA"}, {"offset": 3840, "bold": true, "color": "YELLOW"}]}]}

```

Reply:
Because it's impossible to read this, this is adding a newline at the end of the file.